<!DOCTYPE html>
<html lang="en" class="h-full">
    <head>
        <meta charset="UTF-8">
        <meta name="description" content="Astro description">
        <meta name="viewport" content="width=device-width">
        <link rel="icon" type="image/svg+xml" href="/favicon.svg">
        <meta name="generator" content="Astro v2.6.6">
        <title>Istrail Lab | SNPs and Haplotypes</title>
    <link rel="stylesheet" href="/_astro/essays.5ec62e0b.css" /></head>
    <body class="h-full">
        <div class="w-full flex items-center bg-base-300 px-2 py-3">
    <div class="flex-1 px-2 mx-2">
        <a class="no-underline text-black font-bold text-xl px-2 py-2 rounded-lg hover:bg-[#1F293733]" href="/">Istrail Lab</a>
    </div>
    <style>astro-island,astro-slot,astro-static-slot{display:contents}</style><script>(()=>{var e=async t=>{await(await t())()};(self.Astro||(self.Astro={})).only=e;window.dispatchEvent(new Event("astro:only"));})();;(()=>{var c;{let d={0:t=>t,1:t=>JSON.parse(t,o),2:t=>new RegExp(t),3:t=>new Date(t),4:t=>new Map(JSON.parse(t,o)),5:t=>new Set(JSON.parse(t,o)),6:t=>BigInt(t),7:t=>new URL(t),8:t=>new Uint8Array(JSON.parse(t)),9:t=>new Uint16Array(JSON.parse(t)),10:t=>new Uint32Array(JSON.parse(t))},o=(t,r)=>{if(t===""||!Array.isArray(r))return r;let[e,n]=r;return e in d?d[e](n):void 0};customElements.get("astro-island")||customElements.define("astro-island",(c=class extends HTMLElement{constructor(){super(...arguments);this.hydrate=()=>{var l;if(!this.hydrator||!this.isConnected||(l=this.parentElement)!=null&&l.closest("astro-island[ssr]"))return;let r=this.querySelectorAll("astro-slot"),e={},n=this.querySelectorAll("template[data-astro-template]");for(let s of n){let i=s.closest(this.tagName);!i||!i.isSameNode(this)||(e[s.getAttribute("data-astro-template")||"default"]=s.innerHTML,s.remove())}for(let s of r){let i=s.closest(this.tagName);!i||!i.isSameNode(this)||(e[s.getAttribute("name")||"default"]=s.innerHTML)}let a=this.hasAttribute("props")?JSON.parse(this.getAttribute("props"),o):{};this.hydrator(this)(this.Component,a,e,{client:this.getAttribute("client")}),this.removeAttribute("ssr"),window.removeEventListener("astro:hydrate",this.hydrate),window.dispatchEvent(new CustomEvent("astro:hydrate"))}}connectedCallback(){!this.hasAttribute("await-children")||this.firstChild?this.childrenConnectedCallback():new MutationObserver((r,e)=>{e.disconnect(),setTimeout(()=>this.childrenConnectedCallback(),0)}).observe(this,{childList:!0})}async childrenConnectedCallback(){window.addEventListener("astro:hydrate",this.hydrate);let r=this.getAttribute("before-hydration-url");r&&await import(r),this.start()}start(){let r=JSON.parse(this.getAttribute("opts")),e=this.getAttribute("client");if(Astro[e]===void 0){window.addEventListener(`astro:${e}`,()=>this.start(),{once:!0});return}Astro[e](async()=>{let n=this.getAttribute("renderer-url"),[a,{default:l}]=await Promise.all([import(this.getAttribute("component-url")),n?import(n):()=>()=>{}]),s=this.getAttribute("component-export")||"default";if(!s.includes("."))this.Component=a[s];else{this.Component=a;for(let i of s.split("."))this.Component=this.Component[i]}return this.hydrator=l,this.hydrate},r,this)}attributeChangedCallback(){this.hydrate()}},c.observedAttributes=["props"],c))}})();</script><astro-island uid="aIFOr" component-url="/_astro/NavSidebar.63e48d0f.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{}" ssr="" client="only" opts="{&quot;name&quot;:&quot;NavSidebar&quot;,&quot;value&quot;:&quot;react&quot;}"></astro-island>
</div>
        <div class="flex-1">
            <main style="max-width: min(100ch, 100% - 4rem)" class="mx-auto">
                <h2 class="text-3xl font-bold pt-4">SNPs and Haplotypes</h2>
                <h3 class="text-2xl font-bold py-8">Research Summary</h3>
                <div class="grid gap-4">
    <script>(()=>{var e=async t=>{await(await t())()};(self.Astro||(self.Astro={})).load=e;window.dispatchEvent(new Event("astro:load"));})();</script><astro-island uid="14KBor" prefix="r36" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;Combinatorial and statistical prediction of gene expression from haplotype sequence&quot;],&quot;blurb&quot;:[0,&quot;Genome-wide association studies (GWAS) have discovered thousands\n            of significant genetic effects on disease phenotypes. By\n            considering gene expression as the intermediary between genotype\n            and disease phenotype, expression quantitative trait loci\n            studies have interpreted many of these variants by their\n            regulatory effects on gene expression. However, there remains a\n            considerable gap between genotype-to-gene expression association\n            and genotype-to-gene expression prediction. Accurate prediction\n            of gene expression enables gene-based association studies to be\n            performed post hoc for existing GWAS, reduces multiple testing\n            burden, and can prioritize genes for subsequent experimental\n            investigation.&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">Combinatorial and statistical prediction of gene expression from haplotype sequence</h3>Genome-wide association studies (GWAS) have discovered thousands
            of significant genetic effects on disease phenotypes. By
            considering gene expression as the intermediary between genotype
            and disease phenotype, expression quantitative trait loci
            studies have interpreted many of these variants by their
            regulatory effects on gene expression. However, there remains a
            considerable gap between genotype-to-gene expression association
            and genotype-to-gene expression prediction. Accurate prediction
            of gene expression enables gene-based association studies to be
            performed post hoc for existing GWAS, reduces multiple testing
            burden, and can prioritize genes for subsequent experimental
            investigation.<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
      <p>
        Genome-wide association studies (GWAS) have discovered thousands of
        significant genetic effects on disease phenotypes. By considering gene
        expression as the intermediary between genotype and disease phenotype,
        expression quantitative trait loci studies have interpreted many of
        these variants by their regulatory effects on gene expression. However,
        there remains a considerable gap between genotype-to-gene expression
        association and genotype-to-gene expression prediction. Accurate
        prediction of gene expression enables gene-based association studies to
        be performed post hoc for existing GWAS, reduces multiple testing
        burden, and can prioritize genes for subsequent experimental
        investigation.
      </p>
      <p>
        In this work, we develop gene expression prediction methods that relax
        the independence and additivity assumptions between genetic markers.
        First, we consider gene expression prediction from a regression
        perspective and develop the HAPLEXR algorithm which combines haplotype
        clusterings with allelic dosages. Second, we introduce the new gene
        expression classification problem, which focuses on identifying
        expression groups rather than continuous measurements; we formalize the
        selection of an appropriate number of expression groups using the
        principle of maximum entropy. Third, we develop the HAPLEXD algorithm
        that models haplotype sharing with a modified suffix tree data structure
        and computes expression groups by spectral clustering. In both models,
        we penalize model complexity by prioritizing genetic clusters that
        indicate significant effects on expression. We compare HAPLEXR and
        HAPLEXD with three state-of-the-art expression prediction methods and
        two novel logistic regression approaches across five GTEx v8 tissues.
        HAPLEXD exhibits significantly higher classification accuracy overall;
        HAPLEXR shows higher prediction accuracy on approximately half of the
        genes tested and the largest number of best predicted genes (⁠r2>0.1 ⁠)
        among all methods. We show that variant and haplotype features selected
        by HAPLEXR are smaller in size than competing methods (and thus more
        interpretable) and are significantly enriched in functional annotations
        related to gene regulation. These results demonstrate the importance of
        explicitly modeling non-dosage dependent and intragenic epistatic
        effects when predicting expression.
      </p>
    </template></astro-island>
    <astro-island uid="Z2hBIDi" prefix="r37" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;Tumor Haplotype Assembly Algorithms for Cancer Genomics&quot;],&quot;blurb&quot;:[0,&quot;The growing availability of inexpensive high-throughput sequence data is enabling researchers to sequence tumor populations within a single individual at high coverage. But, cancer genome sequence evolution and mutational phenomena like driver mutations and gene fusions are difficult to investigate without first reconstructing tumor haplotype sequences. Haplotype assembly of single individual tumor populations is an exceedingly difficult task complicated by tumor haplotype heterogeneity, tumor or normal cell sequence contamination, polyploidy, and complex patterns of variation. While computational and experimental haplotype phasing of diploid genomes has seen much progress in recent years, haplotype assembly in cancer genomes remains uncharted territory.&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">Tumor Haplotype Assembly Algorithms for Cancer Genomics</h3>The growing availability of inexpensive high-throughput sequence data is enabling researchers to sequence tumor populations within a single individual at high coverage. But, cancer genome sequence evolution and mutational phenomena like driver mutations and gene fusions are difficult to investigate without first reconstructing tumor haplotype sequences. Haplotype assembly of single individual tumor populations is an exceedingly difficult task complicated by tumor haplotype heterogeneity, tumor or normal cell sequence contamination, polyploidy, and complex patterns of variation. While computational and experimental haplotype phasing of diploid genomes has seen much progress in recent years, haplotype assembly in cancer genomes remains uncharted territory.<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
      <p>
        The growing availability of inexpensive high-throughput sequence data is
        enabling researchers to sequence tumor populations within a single
        individual at high coverage. But, cancer genome sequence evolution and
        mutational phenomena like driver mutations and gene fusions are
        difficult to investigate without first reconstructing tumor haplotype
        sequences. Haplotype assembly of single individual tumor populations is
        an exceedingly difficult task complicated by tumor haplotype
        heterogeneity, tumor or normal cell sequence contamination, polyploidy,
        and complex patterns of variation. While computational and experimental
        haplotype phasing of diploid genomes has seen much progress in recent
        years, haplotype assembly in cancer genomes remains uncharted territory.
      </p>
      <p>
        In this work, we describe HapCompass-Tumor a computational modeling and
        algorithmic framework for haplotype assembly of copy number variable
        cancer genomes containing haplotypes at different frequencies and
        complex variation. We extend our polyploid haplotype assembly model and
        present novel algorithms for (1) complex variations, including copy
        number changes, as varying numbers of disjoint paths in an associated
        graph, (2) variable haplotype frequencies and contamination, and (3)
        computation of tumor haplotypes using simple cycles of the compass graph
        which constrain the space of haplotype assembly solutions. The model and
        algorithm are implemented in the software package HapCompass-Tumor which
        is available for download from
        http://www.brown.edu/Research/Istrail_Lab/archive/.
      </p>
    </template></astro-island>
    <astro-island uid="Z2vtBdL" prefix="r38" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;Tractatus: An Exact and Subquadratic Algorithm for Inferring Identical-by-Descent Multi-shared Haplotype Tracts&quot;],&quot;blurb&quot;:[0,&quot;In this work we present graph theoretic algorithms for the\n            identification of all identical-by-descent (IBD) multi-shared\n            haplotype tracts for an m × n haplotype matrix. We introduce\n            Tractatus, an exact algorithm for computing all IBD haplotype tracts\n            in time linear in the size of the input, O(mn).&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">Tractatus: An Exact and Subquadratic Algorithm for Inferring Identical-by-Descent Multi-shared Haplotype Tracts</h3>In this work we present graph theoretic algorithms for the
            identification of all identical-by-descent (IBD) multi-shared
            haplotype tracts for an m × n haplotype matrix. We introduce
            Tractatus, an exact algorithm for computing all IBD haplotype tracts
            in time linear in the size of the input, O(mn).<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
      In this work we present graph theoretic algorithms for the identification
      of all identical-by-descent (IBD) multi-shared haplotype tracts for an m ×
      n haplotype matrix. We introduce Tractatus, an exact algorithm for
      computing all IBD haplotype tracts in time linear in the size of the
      input, O(mn). Tractatus resolves a long standing open problem, breaking
      optimally the (worst-case) quadratic time barrier of O(m 2 n) of previous
      methods often cited as a bottleneck in haplotype analysis of genome-wide
      association study-sized data. This advance in algorithm efficiency makes
      an impact in a number of areas of population genomics rooted in the
      seminal Li-Stephens framework for modeling multi-loci linkage
      disequilibrium (LD) patterns with applications to the estimation of
      recombination rates, imputation, haplotype-based LD mapping, and haplotype
      phasing. We extend the Tractatus algorithm to include computation of
      haplotype tracts with allele mismatches and shared homozygous haplotypes
      in a set of genotypes. Lastly, we present a comparison of algorithmic
      runtime, power to infer IBD tracts, and false positive rates for simulated
      data and computations of homozygous haplotypes in genome-wide association
      study data of autism. The Tractatus algorithm is available for download at
      http://www.brown.edu/Research/Istrail_lab/archive/ .
    </template></astro-island>
    <astro-island uid="Z1MYnxi" prefix="r39" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;Pathway-based genetic analysis of preterm birth&quot;],&quot;blurb&quot;:[0,&quot;Preterm birth in the United States is now 12%. Multiple genes, gene\n            networks, and variants have been associated with this disease. Using\n            a custom database for preterm birth (dbPTB) with a refined set of\n            genes extensively curated from literature and biological databases,\n            we analyzed GWAS of preterm birth for complete genotype data on\n            nearly 2000 preterm and term mothers.&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">Pathway-based genetic analysis of preterm birth</h3>Preterm birth in the United States is now 12%. Multiple genes, gene
            networks, and variants have been associated with this disease. Using
            a custom database for preterm birth (dbPTB) with a refined set of
            genes extensively curated from literature and biological databases,
            we analyzed GWAS of preterm birth for complete genotype data on
            nearly 2000 preterm and term mothers.<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
      Preterm birth in the United States is now 12%. Multiple genes, gene
      networks, and variants have been associated with this disease. Using a
      custom database for preterm birth (dbPTB) with a refined set of genes
      extensively curated from literature and biological databases, we analyzed
      GWAS of preterm birth for complete genotype data on nearly 2000 preterm
      and term mothers. We used both the curated genes and a genome-wide
      approach to carry out a pathway-based analysis. There were 19 significant
      pathways, which withstood FDR correction for multiple testing that were
      identified using both the curated genes and the genome-wide approach. The
      analysis based on the curated genes was more significant than genome-wide
      in 15 out of 19 pathways. This approach demonstrates the use of a
      validated set of genes, in the analysis of otherwise unsuccessful GWAS
      data, to identify gene–gene interactions in a way that enhances
      statistical power and discovery.
    </template></astro-island>
  </div>
                
  

  <div>
    <h3 class="text-2xl font-bold py-4">
      20 Years of Haplotyping Test of ESA 2021 Test-of-Time Award Time Award For
      Celera Paper
    </h3>
    <p class="pb-8">
      The 30th Annual European Symposium on Algorithms awarded its 2021
      Test-of-Time Award to Sorin Istrail, professor of computational and
      mathematical sciences and professor of computer science, and his
      collaborators for their paper “SNPs Problems, Complexity and Algorithms.”
      The award recognizes “excellent papers in algorithms research” published
      19 to 21 years ago that are “still influential and stimulating for the
      field today,” according to ESA's website.
      <br>
      <a class="hover:text-blue-400 underline" href="https://cs.brown.edu/news/2022/11/16/sorin-istrail-wins-2021-esa-test-time-award/">Read more</a>
    </p>
    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">SNPs Problems, Complexity and Algorithms</h3><span class="text-md font-bold">2001</span></div><p class="py-2">Giuseppe Lancia, Vineet Bafna, Sorin Istrail, Ross Lippert, Russell Schwartz</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>European Symposium on Algorithms</i></span></p></div><div class="flex gap-4 items-center"></div></div></div>
  </div>

  

                

                <h3 class="text-2xl font-bold py-8">Relevant Papers</h3>
                <ul class="grid gap-4">
    <li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Combinatorial and Statistical Predictions of Gene Expression from Haplotype Sequence</h3><span class="text-md font-bold">2020</span></div><p class="py-2">Bert Alpay, Pinar Demetci, Sorin Istrail, Derek Aguiar</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Bioinformatics</i></span></p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="https://academic.oup.com/bioinformatics/article/36/Supplement_1/i194/5870481" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Tumor haplotype assembly algorithms for cancer genomics</h3><span class="text-md font-bold">2014</span></div><p class="py-2">Derek Aguiar, Wendy SW Wong, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Pacific Symposium on Biocomputing</i></span>, vol. 19, pg. 3-14</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.worldscientific.com/doi/pdf/10.1142/9789814583220_0002" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Tractatus: an exact and subquadratic algorithm for inferring identity-by-descent multi-shared haplotype tracts</h3><span class="text-md font-bold">2014</span></div><p class="py-2">Derek Aguiar, Eric Morrow, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>RECOMB</i></span>, pg. 158-173</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="https://link.springer.com/chapter/10.1007/978-3-319-05269-4_1" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Pathway-based genetic analysis of preterm birth</h3><span class="text-md font-bold">2013</span></div><p class="py-2">Alper Uzun, Andrew T. Dewan, Sorin Istrail, James F. Padbury</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Genomics</i></span>, vol. 101, no. 3, pg. 163-170</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.sciencedirect.com/science/article/pii/S0888754312002455" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Haplotype assembly in polyploid genomes and identical by descent shared tracts</h3><span class="text-md font-bold">2013</span></div><p class="py-2">Derek Aguiar, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Bioinformatics</i></span>, vol. 29, no. 13, pg. i352-i360</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://bioinformatics.oxfordjournals.org/content/29/13/i352.abstract" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">HAPCOMPASS: A fast cycle basis algorithm for accurate haplotype assembly of sequence data</h3><span class="text-md font-bold">2012</span></div><p class="py-2">Derek Aguiar, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Journal of Computational Biology</i></span>, vol. 19, no. 6, pg. 577-590</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="https://www.liebertpub.com/doi/epdf/10.1089/cmb.2012.0084" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Global analysis of sequence diversity within HIV-1 subtypes across geographic regions</h3><span class="text-md font-bold">2012</span></div><p class="py-2">Austin Huang, Joseph W Hogan, Sorin Istrail, Allison DeLong, David A Katzenstein, Rami Kantor</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Future Virology</i></span>, vol. 7, no. 5, pg. 505-517</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3400699/pdf/nihms384443.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">DELISHUS: an efficient and exact algorithm for genome-wide detection of deletion polymorphism in autism</h3><span class="text-md font-bold">2012</span></div><p class="py-2">Derek Aguiar, Bjarni V. Halldorsson, Eric M. Morrow, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Bioinformatics</i></span>, vol. 28, no. 12, pg. i154-i162</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="https://watermark.silverchair.com/bioinformatics_28_12_i154.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Conservative Extensions of Linkage Disequilibrium Measures from Pairwise to Multi-loci and Algorithms for Optimal tagsging SNP Selection</h3><span class="text-md font-bold">2011</span></div><p class="py-2">Ryan Tarpine, Fumei Lam, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>Springer Berlin / Heidelberg</p><p><span>In <i>Research in Computational Molecular Biology</i></span>, vol. 6577, pg. 468-482</p></div><div class="flex gap-4 items-center"></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">QColors: An algorithm for conservative viral quasispecies reconstruction from short and non-contiguous next generation sequencing reads</h3><span class="text-md font-bold">2011</span></div><p class="py-2">Austin Huang, Rami Kantor, Allison DeLong, Leeann Schreier, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>IEEE</p><p><span>In <i>BIBM Workshops</i></span>, pg. 130-136</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="/papers/QColors.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The Clark Phaseable Sample Size Problem: Long-Range Phasing and Loss of Heterozygosity in GWAS</h3><span class="text-md font-bold">2011</span></div><p class="py-2">Bjarni Halldorsson, Derek Aguiar, Ryan Tarpine, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Journal of Computational Biology</i></span>, vol. 18, no. 3, pg. 323-333</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="https://www.liebertpub.com/doi/10.1089/cmb.2010.0288" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Haplotype phasing by multi-assembly of shared haplotypes: phase-dependent interactions between rare variants</h3><span class="text-md font-bold">2011</span></div><p class="py-2">Bjarni Halldorsson, Derek Aguiar, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Pac Symp Biocomput.</i></span>, pg. 88-99</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="https://www.worldscientific.com/doi/epdf/10.1142/9789814335058_0010" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The Imperfect Ancestral Recombination Graph Reconstruction Problem: Upper Bounds for Recombination and Homoplasy</h3><span class="text-md font-bold">2010</span></div><p class="py-2">Fumei Lam, Ryan Tarpine, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Journal of Computational Biology</i></span>, vol. 17, no. 6, pg. 767-781</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/impancestralrecomb.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The Clark Phase-able Sample Size Problem: Long-Range Phasing and Loss of Heterozygosity in GWAS</h3><span class="text-md font-bold">2010</span></div><p class="py-2">Bjarni Halldorsson, Derek Aguiar, Ryan Tarpine, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>RECOMB</i></span>, vol. 6044, pg. 158-173</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/clarkphaseable.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Islands of Tractability for Parsimony Haplotyping</h3><span class="text-md font-bold">2006</span></div><p class="py-2">Bjarni V Halldorsson, Roded Sharan, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Stanford Computational Systems Biology</i></span></p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/10.1.1.86.9375.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The Genome of the Sea Urchin Strongylocentrotus purpuratus</h3><span class="text-md font-bold">2006</span></div><p class="py-2">Sea Urchin Genome Sequencing Consortium, Erica Sodergren, George M. Weinstock, Eric H Davidson, R. Andrew Cameron, Richard A. Gibbs, Robert C. Angerer, Lynne M. Angerer, Maria Ina Arnone, David R. Burgess, Robert D. Burke, James A. Coffman, Michael Dean, Maurice R. Elphick, Charles A. Ettensohn, Kathy R. Foltz, Amro Hamdoun, Richard O. Hynes, William H. Klein, William Marzluff, David R. McClay, Robert L. Morris, Arcady Mushegian, Jonathan P. Rast, L. Courtney Smith, Michael C. Thorndyke, Victor D. Vacquier, Gary M. Wessel, Greg Wray, Lan Zhang, Christine G. Elsik, Olga Ermolaeva, Wratko Hlavina, Gretchen Hofmann, Paul Kitts, Melissa J. Landrum, Aaron J. Mackey, Donna Maglott, Georgia Panopoulou, Albert J. Poustka, Kim Pruitt, Victor Sapojnikov, Xingzhi Song, Alexandre Souvorov, Victor Solovyev, Zheng Wei, Charles A. Whittaker, Kim Worley, K. James Durbin, Yufeng Shen, Olivier Fedrigo, David Garfield, Ralph Haygood, Alexander Primus, Rahul Satija, Tonya Severson, Manuel L. Gonzalez-Garay, Andrew R. Jackson, Aleksandar Milosavljevic, Mark Tong, Christopher E. Killian, Brian T. Livingston, Fred H. Wilt, Nikki Adams, Robert Bell, Seth Carbonneau, Rocky Cheung, Patrick Cormier, Bertrand Cosson, Jenifer Croce, Antonio Fernandez-Guerra, Anne-Marie Genevire, Manisha Goel, Hemant Kelkar, Julia Morales, Odile Mulner-Lorillon, Anthony J. Robertson, Jared V. Goldstone, Bryan Cole, David Epel, Bert Gold, Mark E. Hahn, Meredith Howard-Ashby, Mark Scally, John J. Stegeman, Erin L. Allgood, Jonah Cool, Kyle M. Judkins, Shawn S. McCafferty, Ashlan M. Musante, Robert A. Obar, Amanda P. Rawson, Blair J. Rossetti, Ian R. Gibbons, Matthew P. Hoffman, Andrew Leone, Sorin Istrail, Stefan C. Materna, Manoj P. Samanta, Viktor et al. Stolc</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Science</i></span>, vol. 314, no. 5801, pg. 941-952</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/sci.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Gene and alternative splicing annotation with AIR</h3><span class="text-md font-bold">2005</span></div><p class="py-2">Liliana Florea, Valentina Di Francesco, Jason Miller, Russell Turner, Alison Yao, Michael Harris, Brian Walenz, Clark Mobarry, Gennady V. Merkulov, Rosane Charlab, Ian Dew, Zuoming Deng, Sorin Istrail, Peter Li, Granger Sutton</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Genome Research</i></span>, vol. 15, no. 1, pg. 54-66</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/Genome Res.-2005-Florea-54-66.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The linkage disequilibrium maps of three human chromosomes across four populations reflect their demographic history and a common underlying recombination pattern</h3><span class="text-md font-bold">2005</span></div><p class="py-2">Francisco M. De La Vega, Hadar Isaac, Andrew Collins, Charles R. Scafe, Bjarni V. Halldorsson, Xiaoping Su, Ross A. Lippert, Yu Wang, Marion Laig-Webster, Ryan T. Koehler, Janet S. Ziegle, Lewis T. Wogan, Junko F. Stevens, Kyle M. Leinen, Sheri J. Olson, Karl J. Guegler, Xiaoqing You, Lily H. Xu, Heinz G. Hemken, Francis Kalush, Mitsuo Itakura, Yi Zheng, Guy de Thò, Stephen J. O&#x27;Brien, Andrew G. Clark, Sorin Istrail, Michael W. Hunkapiller, Eugene G. Spier, Dennis A. Gilbert</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Genome Research</i></span>, vol. 15, no. 4, pg. 454-462</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/Genome Res.-2005-De La Vega-454-62.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Finding anchors for genomic sequence comparison</h3><span class="text-md font-bold">2004</span></div><p class="py-2">Ross Lippert, Xiaoyue Zhao, Liliana Florea, Clark Mobarry, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>ACM Press</p><p><span>In <i>8th Annual International Conference on Research in Computational Biology (RECOMB04)</i></span></p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/findingAnchors.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Whole Genome Shotgun Assembly and Comparison of Human Genome Assemblies</h3><span class="text-md font-bold">2004</span></div><p class="py-2">Sorin Istrail, Granger G. Sutton, Liliana Florea, Aaron L. Halpern, Clark M. Mobarry, Ross Lippert, Brian Walenz, Hagit Shatkay, Ian Dew, Jason R. Miller, Michael J. Flanigan, Nathan J. Edwards, Randall Bolanos, Daniel Fasulo, Bjarni V. Halldorsson, Sridhar Hannenhalli, Russell Turner, Shibu Yooseph, Fu Lu, Deborah R. Nusskern, Bixiong Chris Shue, Xiangqun Holly Zheng, Fei Zhong, Arthur L. Delcher, Daniel H. Huson, Saul A. Kravitz, Laurent Mouchard, Knut Reinert, Karin A. Remington, Andrew G. Clark, Michael S. Waterman, Evan E. Eichler, Mark D. Adams, Michael W. Hunkapiller, Eugene W. Myers, J. Craig Venter</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Proceedings of the National Academy of Sciences</i></span>, vol. 101, no. 7, pg. 1916-1921</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/PNAS-2004-Istrail-1916-21.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Optimal haplotype block free selection of tagsging SNPs for genome-wide association studies</h3><span class="text-md font-bold">2004</span></div><p class="py-2">Bjarni V Halldorsson, Vineet Bafna, Ross Lippert, Russell Schwartz, Francisco M. De La Vega, Andrew G. Clark, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Genome Research</i></span>, vol. 14, no. 8, pg. 1633-1640</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/Genome Res.-2004-Halldorsson-1633-40.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Robustness of Inference of Haplotype Block Structure</h3><span class="text-md font-bold">2003</span></div><p class="py-2">Russell Schwartz, Bjarni V Halldorsson, Vineet Bafna, Andrew G. Clark, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Journal of Computational Biology</i></span></p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/jcb-03.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Inferring Piecewise Ancestral History from Haploid Sequences</h3><span class="text-md font-bold">2003</span></div><p class="py-2">Russell Schwartz, Andrew G. Clark, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>Springer</p><p><span>In <i>First RECOMB Satellite on Computational Methods for SNPs and Haplotype Inference, (RECOMB SNP&amp;HAP03)</i></span></p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/infpiecewise.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">A survey of computational methods for determining haplotypes</h3><span class="text-md font-bold">2003</span></div><p class="py-2">Bjarni V Halldorsson, Vineet Bafna, Nathan J. Edwards, Ross Lippert, Shibu Yooseph, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>Springer</p><p><span>In <i>First RECOMB Satellite on Computational Methods for SNPs and Haplotype Inference, (RECOMB SNP&amp;HAP03)</i></span>, pg. 26-47</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/IstrailSurvey.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Combinatorial problems arising in SNP and haplotype analysis</h3><span class="text-md font-bold">2003</span></div><p class="py-2">Bjarni V Halldorsson, Vineet Bafna, Nathan J. Edwards, Ross Lippert, Shibu Yooseph, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>Springer</p><p><span>In <i>Fourth International Conference on Discrete Mathematics and Theoretical Computer Science, (CDMTCS03)</i></span>, pg. 26-47</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/10.1.1.123.4929.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Haplotypes and Informative SNP Selection Algorithms: Don&#x27;t Block Out Information</h3><span class="text-md font-bold">2003</span></div><p class="py-2">Vineet Bafna, Bjarni V Halldorsson, Russell Schwartz, Andrew G. Clark, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>ACM Press</p><p><span>In <i>7th Annual International Conference on Computational Molecular Biology (RECOMB03)</i></span>, pg. 19-27</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/informativeSNP.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Methods for inferring block-wise ancestral history from haploid sequences: The haplotype coloring problem</h3><span class="text-md font-bold">2002</span></div><p class="py-2">Russell Schwartz, Andrew G. Clark, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>Springer</p><p><span>In <i>2nd Workshop on Algorithms in Bioinformatics, (WABI02)</i></span>, pg. 44-59</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/methodsinfblock.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Practical Algorithms and Fixed-Parameter Tractability for the Single Individual SNP Haplotyping Problem</h3><span class="text-md font-bold">2002</span></div><p class="py-2">Romeo Rizzi, Vineet Bafna, Sorin Istrail, Giuseppe Lancia</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>Springer</p><p><span>In <i>2nd Workshop on Algorithms in Bioinformatics, (WABI02)</i></span>, pg. 29-43</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/pracAlgsSnpHaplo.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Algorithmic Strategies for the SNP Haplotype Assembly Problem</h3><span class="text-md font-bold">2002</span></div><p class="py-2">Ross Lippert, Russell Schwartz, Giuseppe Lancia, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Briefings in Bioinformatics</i></span>, vol. 3, no. 1, pg. 1-9</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/Brief Bioinform-2002-Lippert-23-31.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The Sequence of the Human Genome</h3><span class="text-md font-bold">2001</span></div><p class="py-2">J. Craig Venter, Mark D. Adams, Eugene W. Myers, Peter W. Li, Richard J. Mural, Granger G. Sutton, Hamilton O. Smith, Mark Yandell, Cheryl A. Evans, Robert A. Holt, Jeannine D. Gocayne, Peter Amanatides, Richard M. Ballew, Daniel H. Huson, Jennifer Russo Wortman, Qing Zhang, Chinnappa D. Kodira, Xiangqun H. Zheng, Lin Chen, Marian Skupski, Gangadharan Subramanian, Paul D. Thomas, Jinghui Zhang, George L. Gabor Miklos, Catherine Nelson, Samuel Broder, Andrew G. Clark, Joe Nadeau, Victor A. McKusick, Norton Zinder, Arnold J. Levine, Richard J. Roberts, Mel Simon, Carolyn Slayman, Michael Hunkapiller, Randall Bolanos, Arthur Delcher, Ian Dew, Daniel Fasulo, Michael Flanigan, Liliana Florea, Aaron Halpern, Sridhar Hannenhalli, Saul Kravitz, Samuel Levy, Clark Mobarry, Knut Reinert, Karin Remington, Jane Abu-Threideh, Ellen Beasley, Kendra Biddick, Vivien Bonazzi, Rhonda Brandon, Michele Cargill, Ishwar Chandramouliswaran, Rosane Charlab, Kabir Chaturvedi, Zuoming Deng, Valentina Di Francesco, Patrick Dunn, Karen Eilbeck, Carlos Evangelista, Andrei E. Gabrielian, Weiniu Gan, Wangmao Ge, Fangcheng Gong, Zhiping Gu, Ping Guan, Thomas J. Heiman, Maureen E. Higgins, Rui-Ru Ji, Zhaoxi Ke, Karen A. Ketchum, Zhongwu Lai, Yiding Lei, Zhenya Li, Jiayin Li, Yong Liang, Xiaoying Lin, Fu Lu, Gennady V. Merkulov, Natalia Milshina, Helen M. Moore, Ashwinikumar K Naik, Vaibhav A. Narayan, Beena Neelam, Deborah Nusskern, Douglas B. Rusch, Steven Salzberg, Wei Shao, Bixiong Shue, Jingtao Sun, Zhen Yuan Wang, Aihui Wang, Xin Wang, Jian Wang, Ming-Hui Wei, Ron Wides, Chunlin Xiao, Chunhua et al. Yan</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Science</i></span>, vol. 291, no. 5507, pg. 1304-1351</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/Science_2001_VenterThe_sequence_of_the_human.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">SNPs Problems: Complexity and Algorithms</h3><span class="text-md font-bold">2001</span></div><p class="py-2">Giuseppe Lancia, Vineet Bafna, Sorin Istrail, Ross Lippert, Russell Schwartz</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>Springer</p><p><span>In <i>EAS</i></span>, vol. 2161, pg. 182-193</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/snpproblems.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Physical Mapping with Repeated Probes: The Hypergraph Superstring Problem</h3><span class="text-md font-bold">1999</span></div><p class="py-2">Serafim Batzoglou, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>Springer</p><p><span>In <i>In 10th Conference on Combinatorial Pattern Matching (CPM99)</i></span></p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/Physical mapping with repeated probes.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">What is a Good Objective for Multiple Sequence Alignment?</h3><span class="text-md font-bold">1998</span></div><p class="py-2">Sorin Istrail, R. Ravi</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Sandia Labs</i></span></p></div><div class="flex gap-4 items-center"></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The Chimeric Mapping Problem: Algorithmic Strategies and Performance Evaluation on Synthetic Genomic Data</h3><span class="text-md font-bold">1994</span></div><p class="py-2">David Greenberg, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Computers and Chemistry</i></span>, vol. 18, no. 3, pg. 207-220</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/The chimeric mapping problem.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The Chimeric Clones Problem</h3><span class="text-md font-bold">1993</span></div><p class="py-2">Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>Sandia Labs</p><p></p></div><div class="flex gap-4 items-center"></div></div></div>
        </li>
  </ul>
            </main>
        </div>
        <footer class="flex justify-center items-center p-4 bg-neutral text-neutral-content mt-20 text-sm">
    <div class="items-center grid-flow-col">
        <p>Copyright Istrail Lab © 2024 - All rights reserved</p>
    </div>
</footer>
    </body></html>