<!DOCTYPE html>
<html lang="en" class="h-full">
    <head>
        <meta charset="UTF-8">
        <meta name="description" content="Astro description">
        <meta name="viewport" content="width=device-width">
        <link rel="icon" type="image/svg+xml" href="/favicon.svg">
        <meta name="generator" content="Astro v2.6.6">
        <title>Istrail Lab | Viruses, Viral Genomics, and Immunopeptidomics</title>
    <link rel="stylesheet" href="/_astro/essays.5ec62e0b.css" /></head>
    <body class="h-full">
        <div class="w-full flex items-center bg-base-300 px-2 py-3">
    <div class="flex-1 px-2 mx-2">
        <a class="no-underline text-black font-bold text-xl px-2 py-2 rounded-lg hover:bg-[#1F293733]" href="/">Istrail Lab</a>
    </div>
    <style>astro-island,astro-slot,astro-static-slot{display:contents}</style><script>(()=>{var e=async t=>{await(await t())()};(self.Astro||(self.Astro={})).only=e;window.dispatchEvent(new Event("astro:only"));})();;(()=>{var c;{let d={0:t=>t,1:t=>JSON.parse(t,o),2:t=>new RegExp(t),3:t=>new Date(t),4:t=>new Map(JSON.parse(t,o)),5:t=>new Set(JSON.parse(t,o)),6:t=>BigInt(t),7:t=>new URL(t),8:t=>new Uint8Array(JSON.parse(t)),9:t=>new Uint16Array(JSON.parse(t)),10:t=>new Uint32Array(JSON.parse(t))},o=(t,r)=>{if(t===""||!Array.isArray(r))return r;let[e,n]=r;return e in d?d[e](n):void 0};customElements.get("astro-island")||customElements.define("astro-island",(c=class extends HTMLElement{constructor(){super(...arguments);this.hydrate=()=>{var l;if(!this.hydrator||!this.isConnected||(l=this.parentElement)!=null&&l.closest("astro-island[ssr]"))return;let r=this.querySelectorAll("astro-slot"),e={},n=this.querySelectorAll("template[data-astro-template]");for(let s of n){let i=s.closest(this.tagName);!i||!i.isSameNode(this)||(e[s.getAttribute("data-astro-template")||"default"]=s.innerHTML,s.remove())}for(let s of r){let i=s.closest(this.tagName);!i||!i.isSameNode(this)||(e[s.getAttribute("name")||"default"]=s.innerHTML)}let a=this.hasAttribute("props")?JSON.parse(this.getAttribute("props"),o):{};this.hydrator(this)(this.Component,a,e,{client:this.getAttribute("client")}),this.removeAttribute("ssr"),window.removeEventListener("astro:hydrate",this.hydrate),window.dispatchEvent(new CustomEvent("astro:hydrate"))}}connectedCallback(){!this.hasAttribute("await-children")||this.firstChild?this.childrenConnectedCallback():new MutationObserver((r,e)=>{e.disconnect(),setTimeout(()=>this.childrenConnectedCallback(),0)}).observe(this,{childList:!0})}async childrenConnectedCallback(){window.addEventListener("astro:hydrate",this.hydrate);let r=this.getAttribute("before-hydration-url");r&&await import(r),this.start()}start(){let r=JSON.parse(this.getAttribute("opts")),e=this.getAttribute("client");if(Astro[e]===void 0){window.addEventListener(`astro:${e}`,()=>this.start(),{once:!0});return}Astro[e](async()=>{let n=this.getAttribute("renderer-url"),[a,{default:l}]=await Promise.all([import(this.getAttribute("component-url")),n?import(n):()=>()=>{}]),s=this.getAttribute("component-export")||"default";if(!s.includes("."))this.Component=a[s];else{this.Component=a;for(let i of s.split("."))this.Component=this.Component[i]}return this.hydrator=l,this.hydrate},r,this)}attributeChangedCallback(){this.hydrate()}},c.observedAttributes=["props"],c))}})();</script><astro-island uid="aIFOr" component-url="/_astro/NavSidebar.63e48d0f.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{}" ssr="" client="only" opts="{&quot;name&quot;:&quot;NavSidebar&quot;,&quot;value&quot;:&quot;react&quot;}"></astro-island>
</div>
        <div class="flex-1">
            <main style="max-width: min(100ch, 100% - 4rem)" class="mx-auto">
                <h2 class="text-3xl font-bold pt-4">Viruses, Viral Genomics, and Immunopeptidomics</h2>
                <h3 class="text-2xl font-bold py-8">Research Summary</h3>
                <div class="grid gap-2">
        <script>(()=>{var e=async t=>{await(await t())()};(self.Astro||(self.Astro={})).load=e;window.dispatchEvent(new Event("astro:load"));})();</script><astro-island uid="ZOJKBb" prefix="r6" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;QColors: An Algorithm for Conservative Viral Quasispecies Reconstruction from Short and Non-Contiguous Next-Generation Sequencing Reads&quot;],&quot;blurb&quot;:[0,&quot;Next generation sequencing technologies have recently been\n            applied to characterize mutational spectra of the heterogeneous\n            population of viral genotypes (known as a quasispecies) within\n            HIV-infected patients. Such information is clinically relevant\n            because minority genetic subpopulations of HIV within patients\n            enable viral escape from selection pressures such as the immune\n            response and antiretroviral therapy. However, methods for\n            quasispecies sequence reconstruction from next generation\n            sequencing reads are not yet widely used and remains an emerging\n            area of research.&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">QColors: An Algorithm for Conservative Viral Quasispecies Reconstruction from Short and Non-Contiguous Next-Generation Sequencing Reads</h3>Next generation sequencing technologies have recently been
            applied to characterize mutational spectra of the heterogeneous
            population of viral genotypes (known as a quasispecies) within
            HIV-infected patients. Such information is clinically relevant
            because minority genetic subpopulations of HIV within patients
            enable viral escape from selection pressures such as the immune
            response and antiretroviral therapy. However, methods for
            quasispecies sequence reconstruction from next generation
            sequencing reads are not yet widely used and remains an emerging
            area of research.<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
            <p>
                Next generation sequencing technologies have recently been
                applied to characterize mutational spectra of the heterogeneous
                population of viral genotypes (known as a quasispecies) within
                HIV-infected patients. Such information is clinically relevant
                because minority genetic subpopulations of HIV within patients
                enable viral escape from selection pressures such as the immune
                response and antiretroviral therapy. However, methods for
                quasispecies sequence reconstruction from next generation
                sequencing reads are not yet widely used and remains an emerging
                area of research. Furthermore, the majority of research
                methodology in HIV has focused on 454 sequencing, while many
                next-generation sequencing platforms used in practice are
                limited to shorter read lengths relative to 454 sequencing.
                Little work has been done in determining how best to address the
                read length limitations of other platforms.
            </p><p>
                The approach described here incorporates graph representations
                of both read differences and read overlap to conservatively
                determine the regions of the sequence with sufficient
                variability to separate quasispecies sequences. Within these
                tractable regions of quasispecies inference, we use constraint
                programming to solve for an optimal quasispecies subsequence
                determination via vertex coloring of the conflict graph, a
                representation which also lends itself to data with
                non-contiguous reads such as paired-end sequencing. We
                demonstrate the utility of the method by applying it to
                simulations based on actual intra-patient clonal HIV-1
                sequencing data.
            </p>
        </template></astro-island>
        <astro-island uid="Z1Agut5" prefix="r7" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;Comparative immunopeptidomics of humans and their pathogens&quot;],&quot;blurb&quot;:[0,&quot;Major histocompatibility complex class I molecules present peptides of 8-10 residues to CD8+ T cells. We used 19 predicted proteomes to determine the influence of CD8+ T cell immune surveillance on protein evolution in humans and microbial &quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">Comparative immunopeptidomics of humans and their pathogens</h3>Major histocompatibility complex class I molecules present peptides of 8-10 residues to CD8+ T cells. We used 19 predicted proteomes to determine the influence of CD8+ T cell immune surveillance on protein evolution in humans and microbial <button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
            Major histocompatibility complex class I molecules present peptides
            of 8-10 residues to CD8+ T cells. We used 19 predicted proteomes to
            determine the influence of CD8+ T cell immune surveillance on
            protein evolution in humans and microbial pathogens by predicting
            immunopeptidomes, i.e., sets of class I binding peptides present in
            proteomes. We find that class I peptide binding specificities (i)
            have had little, if any, influence on the evolution of
            immunopeptidomes and (ii) do not take advantage of biases in amino
            acid distribution in proteins other than the concentration of
            hydrophobic residues in NH2-terminal leader sequences.
        </template></astro-island>
        <astro-island uid="Z1TO3fb" prefix="r8" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;Epitope prediction algorithms for peptide-based vaccine design&quot;],&quot;blurb&quot;:[0,&quot;Peptide-based vaccines, in which small peptides derived from target\n                proteins (eptiopes) are used to provoke an immune reaction, have\n                attracted considerable attention recently as a potential means both\n                of treating infectious diseases and promoting the destruction of\n                cancerous cells by a patient&#39;s own immune system. With the\n                availability of large sequence databases and computers fast enough\n                for rapid processing of large numbers of peptides&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">Epitope prediction algorithms for peptide-based vaccine design</h3>Peptide-based vaccines, in which small peptides derived from target
                proteins (eptiopes) are used to provoke an immune reaction, have
                attracted considerable attention recently as a potential means both
                of treating infectious diseases and promoting the destruction of
                cancerous cells by a patient&#x27;s own immune system. With the
                availability of large sequence databases and computers fast enough
                for rapid processing of large numbers of peptides<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
            Peptide-based vaccines, in which small peptides derived from target
            proteins (eptiopes) are used to provoke an immune reaction, have
            attracted considerable attention recently as a potential means both
            of treating infectious diseases and promoting the destruction of
            cancerous cells by a patient's own immune system. With the
            availability of large sequence databases and computers fast enough
            for rapid processing of large numbers of peptides, computer aided
            design of peptide-based vaccines has emerged as a promising approach
            to screening among billions of possible immune-active peptides to
            find those likely to provoke an immune response to a particular cell
            type. In this paper, we describe the development of three novel
            classes of methods for the prediction problem. We present a
            quadratic programming approach that can be trained on quantitative
            as well as qualitative data. The second method uses linear
            programming to counteract the fact that our training data contains
            mostly positive examples. The third class of methods uses sequence
            profiles obtained by clustering known epitopes to score candidate
            peptides. By integrating these methods, using a simple voting
            heuristic, we achieve improved accuracy over the state of the art.
        </template></astro-island>
        <astro-island uid="Z1tuYvS" prefix="r9" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;The Sequence of the Human Genome&quot;],&quot;blurb&quot;:[0,&quot;A 2.91-billion base pair (bp) consensus sequence of the euchromatic portion of the human genome was generated by the whole-genome shotgun sequencing method. The 14.8-billion bp DNA sequence was generated over 9 months from 27,271,853 high-quality sequence reads (5.11-fold coverage of the genome) from both ends of plasmid clones made from the DNA of five individuals. Two assembly strategies-a whole-genome assembly and a regional chromosome assembly-were used, each combining sequence data from Celera and the publicly funded genome effort.&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">The Sequence of the Human Genome</h3>A 2.91-billion base pair (bp) consensus sequence of the euchromatic portion of the human genome was generated by the whole-genome shotgun sequencing method. The 14.8-billion bp DNA sequence was generated over 9 months from 27,271,853 high-quality sequence reads (5.11-fold coverage of the genome) from both ends of plasmid clones made from the DNA of five individuals. Two assembly strategies-a whole-genome assembly and a regional chromosome assembly-were used, each combining sequence data from Celera and the publicly funded genome effort.<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
            A 2.91-billion base pair (bp) consensus sequence of the euchromatic
            portion of the human genome was generated by the whole-genome
            shotgun sequencing method. The 14.8-billion bp DNA sequence was
            generated over 9 months from 27,271,853 high-quality sequence reads
            (5.11-fold coverage of the genome) from both ends of plasmid clones
            made from the DNA of five individuals. Two assembly strategies-a
            whole-genome assembly and a regional chromosome assembly-were used,
            each combining sequence data from Celera and the publicly funded
            genome effort. The public data were shredded into 550-bp segments to
            create a 2.9-fold coverage of those genome regions that had been
            sequenced, without including biases inherent in the cloning and
            assembly procedure used by the publicly funded group. This brought
            the effective coverage in the assemblies to eightfold, reducing the
            number and size of gaps in the final assembly over what would be
            obtained with 5.11-fold coverage. The two assembly strategies
            yielded very similar results that largely agree with independent
            mapping data. The assemblies effectively cover the euchromatic
            regions of the human chromosomes. More than 90% of the genome is in
            scaffold assemblies of 100,000 bp or more, and 25% of the genome is
            in scaffolds of 10 million bp or larger. Analysis of the genome
            sequence revealed 26,588 protein-encoding transcripts for which
            there was strong corroborating evidence and an additional
            approximately 12,000 computationally derived genes with mouse
            matches or other weak supporting evidence. Although gene-dense
            clusters are obvious, almost half the genes are dispersed in low G+C
            sequence separated by large tracts of apparently noncoding sequence.
            Only 1.1% of the genome is spanned by exons, whereas 24% is in
            introns, with 75% of the genome being intergenic DNA. Duplications
            of segmental blocks, ranging in size up to chromosomal lengths, are
            abundant throughout the genome and reveal a complex evolutionary
            history. Comparative genomic analysis indicates vertebrate
            expansions of genes associated with neuronal function, with
            tissue-specific developmental regulation, and with the hemostasis
            and immune systems. DNA sequence comparisons between the consensus
            sequence and publicly funded genome data provided locations of 2.1
            million single-nucleotide polymorphisms (SNPs). A random pair of
            human haploid genomes differed at a rate of 1 bp per 1250 on
            average, but there was marked heterogeneity in the level of
            polymorphism across the genome. Less than 1% of all SNPs resulted in
            variation in proteins, but the task of determining which SNPs have
            functional consequences remains an open challenge.
        </template></astro-island>
    </div>
                
                

                <h3 class="text-2xl font-bold py-8">Relevant Papers</h3>
                <ul class="grid gap-4">
        <li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">QColors: An algorithm for conservative viral quasispecies reconstruction from short and non-contiguous next generation sequencing reads</h3><span class="text-md font-bold">2011</span></div><p class="py-2">Austin Huang, Rami Kantor, Allison DeLong, Leeann Schreier, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>IEEE</p><p><span>In <i>BIBM Workshops</i></span>, pg. 130-136</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="/papers/QColors.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Comparative Immuno-Peptidomics of Humans and their Pathogens</h3><span class="text-md font-bold">2004</span></div><p class="py-2">Sorin Istrail, Liliana Florea, Bjarni V Halldorsson, Oliver Kohlbacher, Russell Schwartz, Von Bing Yap, Jonathan Yewdell, Stephen Hoffman</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Proceedings of the National Academy of Sciences</i></span>, vol. 101, no. 36, pg. 13268-13272</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/PNAS-2004-Istrail-13268-72.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Epitope prediction algorithms for peptide-based vaccine design</h3><span class="text-md font-bold">2003</span></div><p class="py-2">Liliana Florea, Bjarni V Halldorsson, Oliver Kohlbacher, Russell Schwartz, Stephen Hoffman, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>IEEE Press</p><p><span>In <i>Second Annual IEEE Computer Society Bioinformatics Conference (CSB03)</i></span>, pg. 17-26</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/paper119.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">A Computational Pipeline for High-Throughput T-Cell Epitope Prediction</h3><span class="text-md font-bold">2003</span></div><p class="py-2">Liliana Florea, Bjarni V Halldorsson, Oliver Kohlbacher, Russell Schwartz, Stephen Hoffman, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>Celera Genomics</p><p></p></div><div class="flex gap-4 items-center"></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The Sequence of the Human Genome</h3><span class="text-md font-bold">2001</span></div><p class="py-2">J. Craig Venter, Mark D. Adams, Eugene W. Myers, Peter W. Li, Richard J. Mural, Granger G. Sutton, Hamilton O. Smith, Mark Yandell, Cheryl A. Evans, Robert A. Holt, Jeannine D. Gocayne, Peter Amanatides, Richard M. Ballew, Daniel H. Huson, Jennifer Russo Wortman, Qing Zhang, Chinnappa D. Kodira, Xiangqun H. Zheng, Lin Chen, Marian Skupski, Gangadharan Subramanian, Paul D. Thomas, Jinghui Zhang, George L. Gabor Miklos, Catherine Nelson, Samuel Broder, Andrew G. Clark, Joe Nadeau, Victor A. McKusick, Norton Zinder, Arnold J. Levine, Richard J. Roberts, Mel Simon, Carolyn Slayman, Michael Hunkapiller, Randall Bolanos, Arthur Delcher, Ian Dew, Daniel Fasulo, Michael Flanigan, Liliana Florea, Aaron Halpern, Sridhar Hannenhalli, Saul Kravitz, Samuel Levy, Clark Mobarry, Knut Reinert, Karin Remington, Jane Abu-Threideh, Ellen Beasley, Kendra Biddick, Vivien Bonazzi, Rhonda Brandon, Michele Cargill, Ishwar Chandramouliswaran, Rosane Charlab, Kabir Chaturvedi, Zuoming Deng, Valentina Di Francesco, Patrick Dunn, Karen Eilbeck, Carlos Evangelista, Andrei E. Gabrielian, Weiniu Gan, Wangmao Ge, Fangcheng Gong, Zhiping Gu, Ping Guan, Thomas J. Heiman, Maureen E. Higgins, Rui-Ru Ji, Zhaoxi Ke, Karen A. Ketchum, Zhongwu Lai, Yiding Lei, Zhenya Li, Jiayin Li, Yong Liang, Xiaoying Lin, Fu Lu, Gennady V. Merkulov, Natalia Milshina, Helen M. Moore, Ashwinikumar K Naik, Vaibhav A. Narayan, Beena Neelam, Deborah Nusskern, Douglas B. Rusch, Steven Salzberg, Wei Shao, Bixiong Shue, Jingtao Sun, Zhen Yuan Wang, Aihui Wang, Xin Wang, Jian Wang, Ming-Hui Wei, Ron Wides, Chunlin Xiao, Chunhua et al. Yan</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Science</i></span>, vol. 291, no. 5507, pg. 1304-1351</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/Science_2001_VenterThe_sequence_of_the_human.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Visualization challenges for a new cyberpharmaceutical computing paradigm</h3><span class="text-md font-bold">2001</span></div><p class="py-2">Russell J. Turner, Kabir Chaturvedi, Nathan J. Edwards, Daniel Fasulo, Aaron L. Halpern, Daniel H. Huson, Oliver Kohlbacher, Jason R. Miller, Knut Reinert, Karin A. Remington, Russell Schwartz, Brian Walenz, Shibu Yooseph, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>IEEE Press</p><p><span>In <i>IEEE 2001 symposium on parallel and large-data visualization and graphics</i></span>, pg. 7-18</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/p7-turner.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li>
    </ul>
            </main>
        </div>
        <footer class="flex justify-center items-center p-4 bg-neutral text-neutral-content mt-20 text-sm">
    <div class="items-center grid-flow-col">
        <p>Copyright Istrail Lab © 2024 - All rights reserved</p>
    </div>
</footer>
    </body></html>