<!DOCTYPE html>
<html lang="en" class="h-full">
    <head>
        <meta charset="UTF-8">
        <meta name="description" content="Astro description">
        <meta name="viewport" content="width=device-width">
        <link rel="icon" type="image/svg+xml" href="/favicon.svg">
        <meta name="generator" content="Astro v2.6.6">
        <title>Istrail Lab | Bioinformatics and GWAS</title>
    <link rel="stylesheet" href="/_astro/essays.5ec62e0b.css" /></head>
    <body class="h-full">
        <div class="w-full flex items-center bg-base-300 px-2 py-3">
    <div class="flex-1 px-2 mx-2">
        <a class="no-underline text-black font-bold text-xl px-2 py-2 rounded-lg hover:bg-[#1F293733]" href="/">Istrail Lab</a>
    </div>
    <style>astro-island,astro-slot,astro-static-slot{display:contents}</style><script>(()=>{var e=async t=>{await(await t())()};(self.Astro||(self.Astro={})).only=e;window.dispatchEvent(new Event("astro:only"));})();;(()=>{var c;{let d={0:t=>t,1:t=>JSON.parse(t,o),2:t=>new RegExp(t),3:t=>new Date(t),4:t=>new Map(JSON.parse(t,o)),5:t=>new Set(JSON.parse(t,o)),6:t=>BigInt(t),7:t=>new URL(t),8:t=>new Uint8Array(JSON.parse(t)),9:t=>new Uint16Array(JSON.parse(t)),10:t=>new Uint32Array(JSON.parse(t))},o=(t,r)=>{if(t===""||!Array.isArray(r))return r;let[e,n]=r;return e in d?d[e](n):void 0};customElements.get("astro-island")||customElements.define("astro-island",(c=class extends HTMLElement{constructor(){super(...arguments);this.hydrate=()=>{var l;if(!this.hydrator||!this.isConnected||(l=this.parentElement)!=null&&l.closest("astro-island[ssr]"))return;let r=this.querySelectorAll("astro-slot"),e={},n=this.querySelectorAll("template[data-astro-template]");for(let s of n){let i=s.closest(this.tagName);!i||!i.isSameNode(this)||(e[s.getAttribute("data-astro-template")||"default"]=s.innerHTML,s.remove())}for(let s of r){let i=s.closest(this.tagName);!i||!i.isSameNode(this)||(e[s.getAttribute("name")||"default"]=s.innerHTML)}let a=this.hasAttribute("props")?JSON.parse(this.getAttribute("props"),o):{};this.hydrator(this)(this.Component,a,e,{client:this.getAttribute("client")}),this.removeAttribute("ssr"),window.removeEventListener("astro:hydrate",this.hydrate),window.dispatchEvent(new CustomEvent("astro:hydrate"))}}connectedCallback(){!this.hasAttribute("await-children")||this.firstChild?this.childrenConnectedCallback():new MutationObserver((r,e)=>{e.disconnect(),setTimeout(()=>this.childrenConnectedCallback(),0)}).observe(this,{childList:!0})}async childrenConnectedCallback(){window.addEventListener("astro:hydrate",this.hydrate);let r=this.getAttribute("before-hydration-url");r&&await import(r),this.start()}start(){let r=JSON.parse(this.getAttribute("opts")),e=this.getAttribute("client");if(Astro[e]===void 0){window.addEventListener(`astro:${e}`,()=>this.start(),{once:!0});return}Astro[e](async()=>{let n=this.getAttribute("renderer-url"),[a,{default:l}]=await Promise.all([import(this.getAttribute("component-url")),n?import(n):()=>()=>{}]),s=this.getAttribute("component-export")||"default";if(!s.includes("."))this.Component=a[s];else{this.Component=a;for(let i of s.split("."))this.Component=this.Component[i]}return this.hydrator=l,this.hydrate},r,this)}attributeChangedCallback(){this.hydrate()}},c.observedAttributes=["props"],c))}})();</script><astro-island uid="aIFOr" component-url="/_astro/NavSidebar.63e48d0f.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{}" ssr="" client="only" opts="{&quot;name&quot;:&quot;NavSidebar&quot;,&quot;value&quot;:&quot;react&quot;}"></astro-island>
</div>
        <div class="flex-1">
            <main style="max-width: min(100ch, 100% - 4rem)" class="mx-auto">
                <h2 class="text-3xl font-bold pt-4">Bioinformatics and GWAS</h2>
                <h3 class="text-2xl font-bold py-8">Research Summary</h3>
                <div class="grid gap-4">
        <script>(()=>{var e=async t=>{await(await t())()};(self.Astro||(self.Astro={})).load=e;window.dispatchEvent(new Event("astro:load"));})();</script><astro-island uid="23WngU" prefix="r13" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;Tumor Haplotype Assembly Algorithms for Cancer Genomics&quot;],&quot;blurb&quot;:[0,&quot;The growing availability of inexpensive high-throughput sequence data is enabling researchers to sequence tumor populations within a single individual at high coverage. But, cancer genome sequence evolution and mutational phenomena like driver mutations and gene fusions are difficult to investigate without first reconstructing tumor haplotype sequences. Haplotype assembly of single individual tumor populations is an exceedingly difficult task complicated by tumor haplotype heterogeneity, tumor or normal cell sequence contamination, polyploidy, and complex patterns of variation. While computational and experimental haplotype phasing of diploid genomes has seen much progress in recent years, haplotype assembly in cancer genomes remains uncharted territory.&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">Tumor Haplotype Assembly Algorithms for Cancer Genomics</h3>The growing availability of inexpensive high-throughput sequence data is enabling researchers to sequence tumor populations within a single individual at high coverage. But, cancer genome sequence evolution and mutational phenomena like driver mutations and gene fusions are difficult to investigate without first reconstructing tumor haplotype sequences. Haplotype assembly of single individual tumor populations is an exceedingly difficult task complicated by tumor haplotype heterogeneity, tumor or normal cell sequence contamination, polyploidy, and complex patterns of variation. While computational and experimental haplotype phasing of diploid genomes has seen much progress in recent years, haplotype assembly in cancer genomes remains uncharted territory.<style data-emotion="css 6ii3fu">.css-6ii3fu{font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:500;font-size:0.875rem;line-height:1.75;letter-spacing:0.02857em;text-transform:uppercase;min-width:64px;padding:5px 15px;border-radius:4px;-webkit-transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;border:1px solid rgba(25, 118, 210, 0.5);color:#1976d2;}.css-6ii3fu:hover{-webkit-text-decoration:none;text-decoration:none;background-color:rgba(25, 118, 210, 0.04);border:1px solid #1976d2;}@media (hover: none){.css-6ii3fu:hover{background-color:transparent;}}.css-6ii3fu.Mui-disabled{color:rgba(0, 0, 0, 0.26);border:1px solid rgba(0, 0, 0, 0.12);}</style><style data-emotion="css 79xub">.css-79xub{display:-webkit-inline-box;display:-webkit-inline-flex;display:-ms-inline-flexbox;display:inline-flex;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;-webkit-box-pack:center;-ms-flex-pack:center;-webkit-justify-content:center;justify-content:center;position:relative;box-sizing:border-box;-webkit-tap-highlight-color:transparent;background-color:transparent;outline:0;border:0;margin:0;border-radius:0;padding:0;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;vertical-align:middle;-moz-appearance:none;-webkit-appearance:none;-webkit-text-decoration:none;text-decoration:none;color:inherit;font-family:"Roboto","Helvetica","Arial",sans-serif;font-weight:500;font-size:0.875rem;line-height:1.75;letter-spacing:0.02857em;text-transform:uppercase;min-width:64px;padding:5px 15px;border-radius:4px;-webkit-transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;transition:background-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,box-shadow 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,border-color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,color 250ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;border:1px solid rgba(25, 118, 210, 0.5);color:#1976d2;}.css-79xub::-moz-focus-inner{border-style:none;}.css-79xub.Mui-disabled{pointer-events:none;cursor:default;}@media print{.css-79xub{-webkit-print-color-adjust:exact;color-adjust:exact;}}.css-79xub:hover{-webkit-text-decoration:none;text-decoration:none;background-color:rgba(25, 118, 210, 0.04);border:1px solid #1976d2;}@media (hover: none){.css-79xub:hover{background-color:transparent;}}.css-79xub.Mui-disabled{color:rgba(0, 0, 0, 0.26);border:1px solid rgba(0, 0, 0, 0.12);}</style><button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
            <p>
                The growing availability of inexpensive high-throughput sequence
                data is enabling researchers to sequence tumor populations
                within a single individual at high coverage. But, cancer genome
                sequence evolution and mutational phenomena like driver
                mutations and gene fusions are difficult to investigate without
                first reconstructing tumor haplotype sequences. Haplotype
                assembly of single individual tumor populations is an
                exceedingly difficult task complicated by tumor haplotype
                heterogeneity, tumor or normal cell sequence contamination,
                polyploidy, and complex patterns of variation. While
                computational and experimental haplotype phasing of diploid
                genomes has seen much progress in recent years, haplotype
                assembly in cancer genomes remains uncharted territory.
            </p>
            <p>
                In this work, we describe HapCompass-Tumor a computational
                modeling and algorithmic framework for haplotype assembly of
                copy number variable cancer genomes containing haplotypes at
                different frequencies and complex variation. We extend our
                polyploid haplotype assembly model and present novel algorithms
                for (1) complex variations, including copy number changes, as
                varying numbers of disjoint paths in an associated graph, (2)
                variable haplotype frequencies and contamination, and (3)
                computation of tumor haplotypes using simple cycles of the
                compass graph which constrain the space of haplotype assembly
                solutions. The model and algorithm are implemented in the
                software package HapCompass-Tumor which is available for
                download from http://www.brown.edu/Research/Istrail_Lab/.
            </p>
        </template></astro-island>
        <astro-island uid="y8nN3" prefix="r14" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;Intellectual Disability Is Associated with Increased Runs of Homozygosity in Simplex Autism&quot;],&quot;blurb&quot;:[0,&quot;Intellectual disability (ID), often attributed to\n            autosomal-recessive mutations, occurs in 40% of autism spectrum\n            disorders (ASDs). For this reason, we conducted a genome-wide\n            analysis of runs of homozygosity (ROH) in simplex ASD-affected\n            families consisting of a proband diagnosed with ASD and at least one\n            unaffected sibling.&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">Intellectual Disability Is Associated with Increased Runs of Homozygosity in Simplex Autism</h3>Intellectual disability (ID), often attributed to
            autosomal-recessive mutations, occurs in 40% of autism spectrum
            disorders (ASDs). For this reason, we conducted a genome-wide
            analysis of runs of homozygosity (ROH) in simplex ASD-affected
            families consisting of a proband diagnosed with ASD and at least one
            unaffected sibling.<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
            Intellectual disability (ID), often attributed to
            autosomal-recessive mutations, occurs in 40% of autism spectrum
            disorders (ASDs). For this reason, we conducted a genome-wide
            analysis of runs of homozygosity (ROH) in simplex ASD-affected
            families consisting of a proband diagnosed with ASD and at least one
            unaffected sibling. In these families, probands with an IQ ≤ 70 show
            more ROH than their unaffected siblings, whereas probands with an IQ
            > 70 do not show this excess. Although ASD is far more common in
            males than in females, the proportion of females increases with
            decreasing IQ. Our data do support an association between ROH burden
            and autism diagnosis in girls; however, we are not able to show that
            this effect is independent of low IQ. We have also discovered
            several autism candidate genes on the basis of finding (1) a single
            gene that is within an ROH interval and that is recurrent in autism
            or (2) a gene that is within an autism ROH block and that harbors a
            homozygous, rare deleterious variant upon analysis of
            exome-sequencing data. In summary, our data suggest a distinct
            genetic architecture for participants with autism and co-occurring
            intellectual disability and that this architecture could involve a
            role for recessively inherited loci for this autism subgroup.
        </template></astro-island>
        <astro-island uid="1Cst32" prefix="r15" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;A quantitative reference transcriptome for Nematostella vectensis earlyembryonic development: a pipeline for de novo assembly in emergingmodel systems&quot;],&quot;blurb&quot;:[0,&quot;The de novo assembly of transcriptomes from short shotgun sequencesraises challenges due to random and non-random sequencing biases andinherent transcript complexity. We sought to define a pipeline for denovo transcriptome assembly to aid researchers working withemerging model systems where well annotated genome assemblies are notavailable as a reference.&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">A quantitative reference transcriptome for Nematostella vectensis earlyembryonic development: a pipeline for de novo assembly in emergingmodel systems</h3>The de novo assembly of transcriptomes from short shotgun sequencesraises challenges due to random and non-random sequencing biases andinherent transcript complexity. We sought to define a pipeline for denovo transcriptome assembly to aid researchers working withemerging model systems where well annotated genome assemblies are notavailable as a reference.<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
            <p>
                The de novo assembly of transcriptomes from short shotgun
                sequencesraises challenges due to random and non-random
                sequencing biases andinherent transcript complexity. We sought
                to define a pipeline for denovo transcriptome assembly to aid
                researchers working withemerging model systems where well
                annotated genome assemblies are notavailable as a reference. To
                detail this experimental and computationalmethod, we used early
                embryos of the sea anemone, Nematostellavectensis, an emerging
                model system for studies of animal body planevolution. We
                performed RNA-seq on embryos up to 24 h of developmentusing
                Illumina HiSeq technology and evaluated independent de
                novoassembly methods. The resulting reads were assembled using
                either theTrinity assembler on all quality controlled reads or
                both the Velvet andOases assemblers on reads passing a stringent
                digital normalization filter.A control set of mRNA standards
                from the National Institute of Standards andTechnology (NIST)
                was included in our experimental pipeline to invest
                ourtranscriptome with quantitative information on absolute
                transcript levelsand to provide additional quality control.
            </p>
            <p>
                We generated >200 million paired-end reads from directional cDNA
                librariesrepresenting well over 20 Gb of sequence. The Trinity
                assembler pipeline,including preliminary quality control steps,
                resulted in more than 86% ofreads aligning with the reference
                transcriptome thus generated.Nevertheless, digital normalization
                combined with assembly by Velvet andOases required far less
                computing power and decreased processing time whilestill mapping
                82% of reads. We have made the raw sequencing reads andassembled
                transcriptome publically available.
            </p>
            <p>
                Nematostella vectensis was chosen for its strategic position in
                thetree of life for studies into the origins of the animal body
                plan, however, the challenge of reference-free transcriptome
                assembly is relevant to allsystems for which well annotated gene
                models and independently verifiedgenome assembly may not be
                available. To navigate this new territory, wehave constructed a
                pipeline for library preparation and computationalanalysis for
                de novo transcriptome assembly. The gene modelsdefined by this
                reference transcriptome define the set of genes transcribedin
                early Nematostella development and will provide a valuable
                dataset for further gene regulatory network investigations.
            </p>
        </template></astro-island>
        <astro-island uid="Z2cHSNj" prefix="r16" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;Haplotype assembly in polyploid genomes and identical by descent shared tracts&quot;],&quot;blurb&quot;:[0,&quot;The de novo assembly of transcriptomes from short shotgun sequencesraises challenges due to random and non-random sequencing biases andinherent transcript complexity. We sought to define a pipeline for denovo transcriptome assembly to aid researchers working withemerging model systems where well annotated genome assemblies are notavailable as a reference.&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">Haplotype assembly in polyploid genomes and identical by descent shared tracts</h3>The de novo assembly of transcriptomes from short shotgun sequencesraises challenges due to random and non-random sequencing biases andinherent transcript complexity. We sought to define a pipeline for denovo transcriptome assembly to aid researchers working withemerging model systems where well annotated genome assemblies are notavailable as a reference.<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
            <p>
                Genome-wide haplotype reconstruction from sequence data, or
                haplotype assembly, is at the center of major challenges in
                molecular biology and life sciences. For complex eukaryotic
                organisms like humans, the genome is vast and the population
                samples are growing so rapidly that algorithms processing
                high-throughput sequencing data must scale favorably in terms of
                both accuracy and computational efficiency. Furthermore, current
                models and methodologies for haplotype assembly (i) do not
                consider individuals sharing haplotypes jointly, which reduces
                the size and accuracy of assembled haplotypes, and (ii) are
                unable to model genomes having more than two sets of homologous
                chromosomes (polyploidy). Polyploid organisms are increasingly
                becoming the target of many research groups interested in the
                genomics of disease, phylogenetics, botany and evolution but
                there is an absence of theory and methods for polyploid
                haplotype reconstruction.
            </p>
            <p>
                In this work, we present a number of results, extensions and
                generalizations of compass graphs and our HapCompass framework.
                We prove the theoretical complexity of two haplotype assembly
                optimizations, thereby motivating the use of heuristics.
                Furthermore, we present graph theory–based algorithms for the
                problem of haplotype assembly using our previously developed
                HapCompass framework for (i) novel implementations of haplotype
                assembly optimizations (minimum error correction), (ii) assembly
                of a pair of individuals sharing a haplotype tract identical by
                descent and (iii) assembly of polyploid genomes. We evaluate our
                methods on 1000 Genomes Project, Pacific Biosciences and
                simulated sequence data.
            </p>
        </template></astro-island>
    </div>
                
                

                <h3 class="text-2xl font-bold py-8">Relevant Papers</h3>
                <ul class="grid gap-4">
        <li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Tumor haplotype assembly algorithms for cancer genomics</h3><span class="text-md font-bold">2014</span></div><p class="py-2">Derek Aguiar, Wendy SW Wong, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Pacific Symposium on Biocomputing</i></span>, vol. 19, pg. 3-14</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.worldscientific.com/doi/pdf/10.1142/9789814583220_0002" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Intellectual Disability Is Associated with Increased Runs of Homozygosity in Simplex Autism</h3><span class="text-md font-bold">2013</span></div><p class="py-2">Ece D Gamsiz, Emma W Viscidi, Abbie M Frederick, Shailender Nagpal, Stephan J Sanders, Michael T Murtha, Michael Schmidt, Elizabeth W Triche, Daniel H Geschwind, Matthew W State, others</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>American Journal of Human Genetics</i></span>, vol. vol. 93, no. 1, pp. 103-109</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="https://www.sciencedirect.com/science/article/pii/S0002929713002723?via%3Dihub" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">A quantitative reference transcriptome for Nematostella vectensis early embryonic development: a pipeline for de novo assembly in emerging model systems</h3><span class="text-md font-bold">2013</span></div><p class="py-2">Sarah Tulin, Derek Aguiar, Sorin Istrail, Joel Smith</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>EvoDevo</i></span>, vol. 4, no. 16</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.evodevojournal.com/content/4/1/16/abstract" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Haplotype assembly in polyploid genomes and identical by descent shared tracts</h3><span class="text-md font-bold">2013</span></div><p class="py-2">Derek Aguiar, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Bioinformatics</i></span>, vol. 29, no. 13, pg. i352-i360</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://bioinformatics.oxfordjournals.org/content/29/13/i352.abstract" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">dbPTB: a database for preterm birth</h3><span class="text-md font-bold">2012</span></div><p class="py-2">Alper Uzun, Alyse Laliberte, Jeremy Parker, Caroline Andrew, Emily Winterrowd, Surendra Sharma, Sorin Istrail, James F. Padbury</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Database (Oxford)</i></span>, vol. 2012, pg. bar069</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="https://watermark.silverchair.com/bar069.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">DELISHUS: an efficient and exact algorithm for genome-wide detection of deletion polymorphism in autism</h3><span class="text-md font-bold">2012</span></div><p class="py-2">Derek Aguiar, Bjarni V. Halldorsson, Eric M. Morrow, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Bioinformatics</i></span>, vol. 28, no. 12, pg. i154-i162</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="https://watermark.silverchair.com/bioinformatics_28_12_i154.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">QColors: An algorithm for conservative viral quasispecies reconstruction from short and non-contiguous next generation sequencing reads</h3><span class="text-md font-bold">2011</span></div><p class="py-2">Austin Huang, Rami Kantor, Allison DeLong, Leeann Schreier, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>IEEE</p><p><span>In <i>BIBM Workshops</i></span>, pg. 130-136</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="/papers/QColors.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Comparative Immuno-Peptidomics of Humans and their Pathogens</h3><span class="text-md font-bold">2004</span></div><p class="py-2">Sorin Istrail, Liliana Florea, Bjarni V Halldorsson, Oliver Kohlbacher, Russell Schwartz, Von Bing Yap, Jonathan Yewdell, Stephen Hoffman</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Proceedings of the National Academy of Sciences</i></span>, vol. 101, no. 36, pg. 13268-13272</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/PNAS-2004-Istrail-13268-72.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Epitope prediction algorithms for peptide-based vaccine design</h3><span class="text-md font-bold">2003</span></div><p class="py-2">Liliana Florea, Bjarni V Halldorsson, Oliver Kohlbacher, Russell Schwartz, Stephen Hoffman, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>IEEE Press</p><p><span>In <i>Second Annual IEEE Computer Society Bioinformatics Conference (CSB03)</i></span>, pg. 17-26</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/paper119.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">A Computational Pipeline for High-Throughput T-Cell Epitope Prediction</h3><span class="text-md font-bold">2003</span></div><p class="py-2">Liliana Florea, Bjarni V Halldorsson, Oliver Kohlbacher, Russell Schwartz, Stephen Hoffman, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>Celera Genomics</p><p></p></div><div class="flex gap-4 items-center"></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Visualization challenges for a new cyberpharmaceutical computing paradigm</h3><span class="text-md font-bold">2001</span></div><p class="py-2">Russell J. Turner, Kabir Chaturvedi, Nathan J. Edwards, Daniel Fasulo, Aaron L. Halpern, Daniel H. Huson, Oliver Kohlbacher, Jason R. Miller, Knut Reinert, Karin A. Remington, Russell Schwartz, Brian Walenz, Shibu Yooseph, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>IEEE Press</p><p><span>In <i>IEEE 2001 symposium on parallel and large-data visualization and graphics</i></span>, pg. 7-18</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/p7-turner.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Algorithmic Strategies in Combinatorial Chemistry</h3><span class="text-md font-bold">2000</span></div><p class="py-2">Debbie Goldman, Sorin Istrail, Giuseppe Lancia, Antonio Piccolboni, Brian Walenz</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>ACM Press</p><p><span>In <i>SODA</i></span></p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/p275-goldman.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Insights Into the Association of Partially Folded Chains Derived From Lattice Simulation Models</h3><span class="text-md font-bold">1999</span></div><p class="py-2">Russell Schwartz, Sorin Istrail, Jonathan King</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>Sandia Labs</p><p></p></div><div class="flex gap-4 items-center"></div></div></div>
                </li>
    </ul>
            </main>
        </div>
        <footer class="flex justify-center items-center p-4 bg-neutral text-neutral-content mt-20 text-sm">
    <div class="items-center grid-flow-col">
        <p>Copyright Istrail Lab © 2024 - All rights reserved</p>
    </div>
</footer>
    </body></html>