<!DOCTYPE html>
<html lang="en" class="h-full">
    <head>
        <meta charset="UTF-8">
        <meta name="description" content="Astro description">
        <meta name="viewport" content="width=device-width">
        <link rel="icon" type="image/svg+xml" href="/favicon.svg">
        <meta name="generator" content="Astro v2.6.6">
        <title>Istrail Lab | Genome Assembly and Comparisons, Haplotype Assembly, and Transcriptome Assembly</title>
    <link rel="stylesheet" href="/_astro/essays.5ec62e0b.css" /></head>
    <body class="h-full">
        <div class="w-full flex items-center bg-base-300 px-2 py-3">
    <div class="flex-1 px-2 mx-2">
        <a class="no-underline text-black font-bold text-xl px-2 py-2 rounded-lg hover:bg-[#1F293733]" href="/">Istrail Lab</a>
    </div>
    <style>astro-island,astro-slot,astro-static-slot{display:contents}</style><script>(()=>{var e=async t=>{await(await t())()};(self.Astro||(self.Astro={})).only=e;window.dispatchEvent(new Event("astro:only"));})();;(()=>{var c;{let d={0:t=>t,1:t=>JSON.parse(t,o),2:t=>new RegExp(t),3:t=>new Date(t),4:t=>new Map(JSON.parse(t,o)),5:t=>new Set(JSON.parse(t,o)),6:t=>BigInt(t),7:t=>new URL(t),8:t=>new Uint8Array(JSON.parse(t)),9:t=>new Uint16Array(JSON.parse(t)),10:t=>new Uint32Array(JSON.parse(t))},o=(t,r)=>{if(t===""||!Array.isArray(r))return r;let[e,n]=r;return e in d?d[e](n):void 0};customElements.get("astro-island")||customElements.define("astro-island",(c=class extends HTMLElement{constructor(){super(...arguments);this.hydrate=()=>{var l;if(!this.hydrator||!this.isConnected||(l=this.parentElement)!=null&&l.closest("astro-island[ssr]"))return;let r=this.querySelectorAll("astro-slot"),e={},n=this.querySelectorAll("template[data-astro-template]");for(let s of n){let i=s.closest(this.tagName);!i||!i.isSameNode(this)||(e[s.getAttribute("data-astro-template")||"default"]=s.innerHTML,s.remove())}for(let s of r){let i=s.closest(this.tagName);!i||!i.isSameNode(this)||(e[s.getAttribute("name")||"default"]=s.innerHTML)}let a=this.hasAttribute("props")?JSON.parse(this.getAttribute("props"),o):{};this.hydrator(this)(this.Component,a,e,{client:this.getAttribute("client")}),this.removeAttribute("ssr"),window.removeEventListener("astro:hydrate",this.hydrate),window.dispatchEvent(new CustomEvent("astro:hydrate"))}}connectedCallback(){!this.hasAttribute("await-children")||this.firstChild?this.childrenConnectedCallback():new MutationObserver((r,e)=>{e.disconnect(),setTimeout(()=>this.childrenConnectedCallback(),0)}).observe(this,{childList:!0})}async childrenConnectedCallback(){window.addEventListener("astro:hydrate",this.hydrate);let r=this.getAttribute("before-hydration-url");r&&await import(r),this.start()}start(){let r=JSON.parse(this.getAttribute("opts")),e=this.getAttribute("client");if(Astro[e]===void 0){window.addEventListener(`astro:${e}`,()=>this.start(),{once:!0});return}Astro[e](async()=>{let n=this.getAttribute("renderer-url"),[a,{default:l}]=await Promise.all([import(this.getAttribute("component-url")),n?import(n):()=>()=>{}]),s=this.getAttribute("component-export")||"default";if(!s.includes("."))this.Component=a[s];else{this.Component=a;for(let i of s.split("."))this.Component=this.Component[i]}return this.hydrator=l,this.hydrate},r,this)}attributeChangedCallback(){this.hydrate()}},c.observedAttributes=["props"],c))}})();</script><astro-island uid="aIFOr" component-url="/_astro/NavSidebar.63e48d0f.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{}" ssr="" client="only" opts="{&quot;name&quot;:&quot;NavSidebar&quot;,&quot;value&quot;:&quot;react&quot;}"></astro-island>
</div>
        <div class="flex-1">
            <main style="max-width: min(100ch, 100% - 4rem)" class="mx-auto">
                <h2 class="text-3xl font-bold pt-4">Genome Assembly and Comparisons, Haplotype Assembly, and Transcriptome Assembly</h2>
                <h3 class="text-2xl font-bold py-8">Research Summary</h3>
                <div class="grid gap-4">
    <script>(()=>{var e=async t=>{await(await t())()};(self.Astro||(self.Astro={})).load=e;window.dispatchEvent(new Event("astro:load"));})();</script><astro-island uid="2uI3N3" prefix="r9" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;blurb&quot;:[0,&quot;The American oyster Crassostrea virginica, an ecologically and\n                economically important estuarine organism, can suffer high\n                mortalities in areas in the Northeast United States due to\n                Roseovarius Oyster Disease (ROD), caused by the gram-negative\n                bacterial pathogen Roseovarius crassostreae. The goals of this\n                research were to provide insights into:&quot;],&quot;title&quot;:[0,&quot;Transcriptome of American Oysters, Crassostrea virginica, in Response to Bacterial Challenge: Insights into Potential Mechanisms of Disease Resistance&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">Transcriptome of American Oysters, Crassostrea virginica, in Response to Bacterial Challenge: Insights into Potential Mechanisms of Disease Resistance</h3>The American oyster Crassostrea virginica, an ecologically and
                economically important estuarine organism, can suffer high
                mortalities in areas in the Northeast United States due to
                Roseovarius Oyster Disease (ROD), caused by the gram-negative
                bacterial pathogen Roseovarius crassostreae. The goals of this
                research were to provide insights into:<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
      The American oyster Crassostrea virginica, an ecologically and
      economically important estuarine organism, can suffer high mortalities in
      areas in the Northeast United States due to Roseovarius Oyster Disease
      (ROD), caused by the gram-negative bacterial pathogen Roseovarius
      crassostreae. The goals of this research were to provide insights into: 1)
      the responses of American oysters to R. crassostreae, and 2) potential
      mechanisms of resistance or susceptibility to ROD. The responses of
      oysters to bacterial challenge were characterized by exposing oysters from
      ROD-resistant and susceptible families to R. crassostreae, followed by
      high-throughput sequencing of cDNA samples from various timepoints after
      disease challenge. Sequence data was assembled into a reference
      transcriptome and analyzed through differential gene expression and
      functional enrichment to uncover genes and processes potentially involved
      in responses to ROD in the American oyster. While susceptible oysters
      experienced constant levels of mortality when challenged with R.
      crassostreae, resistant oysters showed levels of mortality similar to
      non-challenged oysters. Oysters exposed to R. crassostreae showed
      differential expression of transcripts involved in immune recognition,
      signaling, protease inhibition, detoxification, and apoptosis. Transcripts
      involved in metabolism were enriched in susceptible oysters, suggesting
      that bacterial infection places a large metabolic demand on these oysters.
      Transcripts differentially expressed in resistant oysters in response to
      infection included the immune modulators IL-17 and arginase, as well as
      several genes involved in extracellular matrix remodeling. The
      identification of potential genes and processes responsible for defense
      against R. crassostreae in the American oyster provides insights into
      potential mechanisms of disease resistance.
    </template></astro-island>
    <astro-island uid="1Cst32" prefix="r10" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;A quantitative reference transcriptome for Nematostella vectensis earlyembryonic development: a pipeline for de novo assembly in emergingmodel systems&quot;],&quot;blurb&quot;:[0,&quot;The de novo assembly of transcriptomes from short shotgun sequencesraises challenges due to random and non-random sequencing biases andinherent transcript complexity. We sought to define a pipeline for denovo transcriptome assembly to aid researchers working withemerging model systems where well annotated genome assemblies are notavailable as a reference.&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">A quantitative reference transcriptome for Nematostella vectensis earlyembryonic development: a pipeline for de novo assembly in emergingmodel systems</h3>The de novo assembly of transcriptomes from short shotgun sequencesraises challenges due to random and non-random sequencing biases andinherent transcript complexity. We sought to define a pipeline for denovo transcriptome assembly to aid researchers working withemerging model systems where well annotated genome assemblies are notavailable as a reference.<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
      <p>
        The de novo assembly of transcriptomes from short shotgun
        sequencesraises challenges due to random and non-random sequencing
        biases andinherent transcript complexity. We sought to define a pipeline
        for denovo transcriptome assembly to aid researchers working
        withemerging model systems where well annotated genome assemblies are
        notavailable as a reference. To detail this experimental and
        computationalmethod, we used early embryos of the sea anemone,
        Nematostellavectensis, an emerging model system for studies of animal
        body planevolution. We performed RNA-seq on embryos up to 24 h of
        developmentusing Illumina HiSeq technology and evaluated independent de
        novoassembly methods. The resulting reads were assembled using either
        theTrinity assembler on all quality controlled reads or both the Velvet
        andOases assemblers on reads passing a stringent digital normalization
        filter.A control set of mRNA standards from the National Institute of
        Standards andTechnology (NIST) was included in our experimental pipeline
        to invest ourtranscriptome with quantitative information on absolute
        transcript levelsand to provide additional quality control.
      </p>
      <p>
        We generated >200 million paired-end reads from directional cDNA
        librariesrepresenting well over 20 Gb of sequence. The Trinity assembler
        pipeline,including preliminary quality control steps, resulted in more
        than 86% ofreads aligning with the reference transcriptome thus
        generated.Nevertheless, digital normalization combined with assembly by
        Velvet andOases required far less computing power and decreased
        processing time whilestill mapping 82% of reads. We have made the raw
        sequencing reads andassembled transcriptome publically available.
      </p>
      <p>
        Nematostella vectensis was chosen for its strategic position in thetree
        of life for studies into the origins of the animal body plan, however,
        the challenge of reference-free transcriptome assembly is relevant to
        allsystems for which well annotated gene models and independently
        verifiedgenome assembly may not be available. To navigate this new
        territory, wehave constructed a pipeline for library preparation and
        computationalanalysis for de novo transcriptome assembly. The gene
        modelsdefined by this reference transcriptome define the set of genes
        transcribedin early Nematostella development and will provide a valuable
        dataset for further gene regulatory network investigations.
      </p>
    </template></astro-island>
    <astro-island uid="Z2jCl6r" prefix="r11" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;HapCompass: A Fast Cycle Basis Algorithm forAccurate Haplotype Assembly of Sequence Data&quot;],&quot;blurb&quot;:[0,&quot;\n         Genome assembly methods produce haplotype phase ambiguous assemblies\n            due to limita-tions in current sequencing technologies. Determining\n            the haplotype phase of an individualis computationally challenging\n            and experimentally expensive. However, haplotype phaseinformation is\n            crucial in many bioinformatics workflows such as genetic association\n            studiesand genomic imputation.&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">HapCompass: A Fast Cycle Basis Algorithm forAccurate Haplotype Assembly of Sequence Data</h3>
         Genome assembly methods produce haplotype phase ambiguous assemblies
            due to limita-tions in current sequencing technologies. Determining
            the haplotype phase of an individualis computationally challenging
            and experimentally expensive. However, haplotype phaseinformation is
            crucial in many bioinformatics workflows such as genetic association
            studiesand genomic imputation.<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
      Genome assembly methods produce haplotype phase ambiguous assemblies due
      to limita-tions in current sequencing technologies. Determining the
      haplotype phase of an individualis computationally challenging and
      experimentally expensive. However, haplotype phaseinformation is crucial
      in many bioinformatics workflows such as genetic association studiesand
      genomic imputation. Current computational methods of determining haplotype
      phasefrom sequence data—known as haplotype assembly—have difficulties
      producing accurateresults for large (1000 Genomes-type) data or operate on
      restricted optimizations that areunrealistic considering modern
      high-throughput sequencing technologies. We present anovel algorithm,
      HapCompass, for haplotype assembly of densely sequenced human genomedata.
      The HapCompass algorithm operates on a graph where single nucleotide
      polymor-phisms (SNPs) are nodes and edges are defined by sequence reads
      and viewed as supportingevidence of co-occurring SNP alleles in a
      haplotype. In our graph model, haplotype phasingscorrespond to spanning
      trees. We define the minimum weighted edge removal optimizationon this
      graph and develop an algorithm based on cycle basis local optimizations
      for resolvingconflicting evidence. We then estimate the amount of
      sequencing required to produce acomplete haplotype assembly of a
      chromosome. Using these estimates together with metricsborrowed from
      genome assembly and haplotype phasing, we compare the accuracy
      ofHapCompass, the Genome Analysis ToolKit, and HapCut for 1000 Genomes
      Project andsimulated data. We show that HapCompass performs significantly
      better for a variety ofdata and metrics. HapCompass is freely available
      for download (www.brown.edu/Research/Istrail_Lab/archive/).
    </template></astro-island>
    <astro-island uid="Zhyi6K" prefix="r12" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;The Transcriptome of the Sea Urchin Embryo&quot;],&quot;blurb&quot;:[0,&quot;\n         The sea urchin Strongylocentrotus purpuratus is a model organism for\n            study of the genomic control circuitry underlying embryonic\n            development. We examined the complete repertoire of genes expressed\n            in the S. purpuratus embryo, up to late gastrula stage, by means of\n            high-resolution custom tiling arrays covering the whole genome.&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">The Transcriptome of the Sea Urchin Embryo</h3>
         The sea urchin Strongylocentrotus purpuratus is a model organism for
            study of the genomic control circuitry underlying embryonic
            development. We examined the complete repertoire of genes expressed
            in the S. purpuratus embryo, up to late gastrula stage, by means of
            high-resolution custom tiling arrays covering the whole genome.<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
      The sea urchin Strongylocentrotus purpuratus is a model organism for study
      of the genomic control circuitry underlying embryonic development. We
      examined the complete repertoire of genes expressed in the S. purpuratus
      embryo, up to late gastrula stage, by means of high-resolution custom
      tiling arrays covering the whole genome. We detected complete spliced
      structures even for genes known to be expressed at low levels in only a
      few cells. At least 11,000 to 12,000 genes are used in embryogenesis.
      These include most of the genes encoding transcription factors and
      signaling proteins, as well as some classes of general cytoskeletal and
      metabolic proteins, but only a minor fraction of genes encoding immune
      functions and sensory receptors. Thousands of small asymmetric transcripts
      of unknown function were also detected in intergenic regions throughout
      the genome. The tiling array data were used to correct and authenticate
      several thousand gene models during the genome annotation process.
    </template></astro-island>
  </div>
                
                

                <h3 class="text-2xl font-bold py-8">Relevant Papers</h3>
                <ul class="grid gap-4">
    <li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Transcriptome of American oysters, Crassostrea virginica, in response to bacterial challenge: Insights into potential mechanisms of disease resistance</h3><span class="text-md font-bold">2014</span></div><p class="py-2">Ian C. McDowell, Chamilani Nikapitiya, Derek Aguiar, Christopher E. Lane, Sorin Istrail, Marta Gomez-Chiarri</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>PLOS One</i></span></p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="https://journals.plos.org/plosone/article/file?id=10.1371/journal.pone.0105097&amp;type=printable" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">A quantitative reference transcriptome for Nematostella vectensis early embryonic development: a pipeline for de novo assembly in emerging model systems</h3><span class="text-md font-bold">2013</span></div><p class="py-2">Sarah Tulin, Derek Aguiar, Sorin Istrail, Joel Smith</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>EvoDevo</i></span>, vol. 4, no. 16</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.evodevojournal.com/content/4/1/16/abstract" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">HAPCOMPASS: A fast cycle basis algorithm for accurate haplotype assembly of sequence data</h3><span class="text-md font-bold">2012</span></div><p class="py-2">Derek Aguiar, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Journal of Computational Biology</i></span>, vol. 19, no. 6, pg. 577-590</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="https://www.liebertpub.com/doi/epdf/10.1089/cmb.2012.0084" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The Transcriptome of the Sea Urchin Embryo</h3><span class="text-md font-bold">2006</span></div><p class="py-2">Manoj P. Samanta, Waraporn Tongprasit, Sorin Istrail, R. Andrew Cameron, Qiang Tu, Eric H. Davidson, Viktor Stolc</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Science</i></span>, vol. 314, no. 5801, pg. 960-962</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/transcSeaUrchin.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The Genome of the Sea Urchin Strongylocentrotus purpuratus</h3><span class="text-md font-bold">2006</span></div><p class="py-2">Sea Urchin Genome Sequencing Consortium, Erica Sodergren, George M. Weinstock, Eric H Davidson, R. Andrew Cameron, Richard A. Gibbs, Robert C. Angerer, Lynne M. Angerer, Maria Ina Arnone, David R. Burgess, Robert D. Burke, James A. Coffman, Michael Dean, Maurice R. Elphick, Charles A. Ettensohn, Kathy R. Foltz, Amro Hamdoun, Richard O. Hynes, William H. Klein, William Marzluff, David R. McClay, Robert L. Morris, Arcady Mushegian, Jonathan P. Rast, L. Courtney Smith, Michael C. Thorndyke, Victor D. Vacquier, Gary M. Wessel, Greg Wray, Lan Zhang, Christine G. Elsik, Olga Ermolaeva, Wratko Hlavina, Gretchen Hofmann, Paul Kitts, Melissa J. Landrum, Aaron J. Mackey, Donna Maglott, Georgia Panopoulou, Albert J. Poustka, Kim Pruitt, Victor Sapojnikov, Xingzhi Song, Alexandre Souvorov, Victor Solovyev, Zheng Wei, Charles A. Whittaker, Kim Worley, K. James Durbin, Yufeng Shen, Olivier Fedrigo, David Garfield, Ralph Haygood, Alexander Primus, Rahul Satija, Tonya Severson, Manuel L. Gonzalez-Garay, Andrew R. Jackson, Aleksandar Milosavljevic, Mark Tong, Christopher E. Killian, Brian T. Livingston, Fred H. Wilt, Nikki Adams, Robert Bell, Seth Carbonneau, Rocky Cheung, Patrick Cormier, Bertrand Cosson, Jenifer Croce, Antonio Fernandez-Guerra, Anne-Marie Genevire, Manisha Goel, Hemant Kelkar, Julia Morales, Odile Mulner-Lorillon, Anthony J. Robertson, Jared V. Goldstone, Bryan Cole, David Epel, Bert Gold, Mark E. Hahn, Meredith Howard-Ashby, Mark Scally, John J. Stegeman, Erin L. Allgood, Jonah Cool, Kyle M. Judkins, Shawn S. McCafferty, Ashlan M. Musante, Robert A. Obar, Amanda P. Rawson, Blair J. Rossetti, Ian R. Gibbons, Matthew P. Hoffman, Andrew Leone, Sorin Istrail, Stefan C. Materna, Manoj P. Samanta, Viktor et al. Stolc</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Science</i></span>, vol. 314, no. 5801, pg. 941-952</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/sci.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Whole Genome Shotgun Assembly and Comparison of Human Genome Assemblies</h3><span class="text-md font-bold">2004</span></div><p class="py-2">Sorin Istrail, Granger G. Sutton, Liliana Florea, Aaron L. Halpern, Clark M. Mobarry, Ross Lippert, Brian Walenz, Hagit Shatkay, Ian Dew, Jason R. Miller, Michael J. Flanigan, Nathan J. Edwards, Randall Bolanos, Daniel Fasulo, Bjarni V. Halldorsson, Sridhar Hannenhalli, Russell Turner, Shibu Yooseph, Fu Lu, Deborah R. Nusskern, Bixiong Chris Shue, Xiangqun Holly Zheng, Fei Zhong, Arthur L. Delcher, Daniel H. Huson, Saul A. Kravitz, Laurent Mouchard, Knut Reinert, Karin A. Remington, Andrew G. Clark, Michael S. Waterman, Evan E. Eichler, Mark D. Adams, Michael W. Hunkapiller, Eugene W. Myers, J. Craig Venter</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Proceedings of the National Academy of Sciences</i></span>, vol. 101, no. 7, pg. 1916-1921</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/PNAS-2004-Istrail-1916-21.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The Sequence of the Human Genome</h3><span class="text-md font-bold">2001</span></div><p class="py-2">J. Craig Venter, Mark D. Adams, Eugene W. Myers, Peter W. Li, Richard J. Mural, Granger G. Sutton, Hamilton O. Smith, Mark Yandell, Cheryl A. Evans, Robert A. Holt, Jeannine D. Gocayne, Peter Amanatides, Richard M. Ballew, Daniel H. Huson, Jennifer Russo Wortman, Qing Zhang, Chinnappa D. Kodira, Xiangqun H. Zheng, Lin Chen, Marian Skupski, Gangadharan Subramanian, Paul D. Thomas, Jinghui Zhang, George L. Gabor Miklos, Catherine Nelson, Samuel Broder, Andrew G. Clark, Joe Nadeau, Victor A. McKusick, Norton Zinder, Arnold J. Levine, Richard J. Roberts, Mel Simon, Carolyn Slayman, Michael Hunkapiller, Randall Bolanos, Arthur Delcher, Ian Dew, Daniel Fasulo, Michael Flanigan, Liliana Florea, Aaron Halpern, Sridhar Hannenhalli, Saul Kravitz, Samuel Levy, Clark Mobarry, Knut Reinert, Karin Remington, Jane Abu-Threideh, Ellen Beasley, Kendra Biddick, Vivien Bonazzi, Rhonda Brandon, Michele Cargill, Ishwar Chandramouliswaran, Rosane Charlab, Kabir Chaturvedi, Zuoming Deng, Valentina Di Francesco, Patrick Dunn, Karen Eilbeck, Carlos Evangelista, Andrei E. Gabrielian, Weiniu Gan, Wangmao Ge, Fangcheng Gong, Zhiping Gu, Ping Guan, Thomas J. Heiman, Maureen E. Higgins, Rui-Ru Ji, Zhaoxi Ke, Karen A. Ketchum, Zhongwu Lai, Yiding Lei, Zhenya Li, Jiayin Li, Yong Liang, Xiaoying Lin, Fu Lu, Gennady V. Merkulov, Natalia Milshina, Helen M. Moore, Ashwinikumar K Naik, Vaibhav A. Narayan, Beena Neelam, Deborah Nusskern, Douglas B. Rusch, Steven Salzberg, Wei Shao, Bixiong Shue, Jingtao Sun, Zhen Yuan Wang, Aihui Wang, Xin Wang, Jian Wang, Ming-Hui Wei, Ron Wides, Chunlin Xiao, Chunhua et al. Yan</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Science</i></span>, vol. 291, no. 5507, pg. 1304-1351</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/Science_2001_VenterThe_sequence_of_the_human.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Visualization challenges for a new cyberpharmaceutical computing paradigm</h3><span class="text-md font-bold">2001</span></div><p class="py-2">Russell J. Turner, Kabir Chaturvedi, Nathan J. Edwards, Daniel Fasulo, Aaron L. Halpern, Daniel H. Huson, Oliver Kohlbacher, Jason R. Miller, Knut Reinert, Karin A. Remington, Russell Schwartz, Brian Walenz, Shibu Yooseph, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>IEEE Press</p><p><span>In <i>IEEE 2001 symposium on parallel and large-data visualization and graphics</i></span>, pg. 7-18</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/p7-turner.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li><li>
          <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The Chimeric Mapping Problem: Algorithmic Strategies and Performance Evaluation on Synthetic Genomic Data</h3><span class="text-md font-bold">1994</span></div><p class="py-2">David Greenberg, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Computers and Chemistry</i></span>, vol. 18, no. 3, pg. 207-220</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/The chimeric mapping problem.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
        </li>
  </ul>
            </main>
        </div>
        <footer class="flex justify-center items-center p-4 bg-neutral text-neutral-content mt-20 text-sm">
    <div class="items-center grid-flow-col">
        <p>Copyright Istrail Lab © 2024 - All rights reserved</p>
    </div>
</footer>
    </body></html>