<!DOCTYPE html>
<html lang="en" class="h-full">
    <head>
        <meta charset="UTF-8">
        <meta name="description" content="Astro description">
        <meta name="viewport" content="width=device-width">
        <link rel="icon" type="image/svg+xml" href="/favicon.svg">
        <meta name="generator" content="Astro v2.6.6">
        <title>Istrail Lab | The Regulatory Genome, Gene Regulatory Networks and Transcriptomics</title>
    <link rel="stylesheet" href="/_astro/essays.5ec62e0b.css" /></head>
    <body class="h-full">
        <div class="w-full flex items-center bg-base-300 px-2 py-3">
    <div class="flex-1 px-2 mx-2">
        <a class="no-underline text-black font-bold text-xl px-2 py-2 rounded-lg hover:bg-[#1F293733]" href="/">Istrail Lab</a>
    </div>
    <style>astro-island,astro-slot,astro-static-slot{display:contents}</style><script>(()=>{var e=async t=>{await(await t())()};(self.Astro||(self.Astro={})).only=e;window.dispatchEvent(new Event("astro:only"));})();;(()=>{var c;{let d={0:t=>t,1:t=>JSON.parse(t,o),2:t=>new RegExp(t),3:t=>new Date(t),4:t=>new Map(JSON.parse(t,o)),5:t=>new Set(JSON.parse(t,o)),6:t=>BigInt(t),7:t=>new URL(t),8:t=>new Uint8Array(JSON.parse(t)),9:t=>new Uint16Array(JSON.parse(t)),10:t=>new Uint32Array(JSON.parse(t))},o=(t,r)=>{if(t===""||!Array.isArray(r))return r;let[e,n]=r;return e in d?d[e](n):void 0};customElements.get("astro-island")||customElements.define("astro-island",(c=class extends HTMLElement{constructor(){super(...arguments);this.hydrate=()=>{var l;if(!this.hydrator||!this.isConnected||(l=this.parentElement)!=null&&l.closest("astro-island[ssr]"))return;let r=this.querySelectorAll("astro-slot"),e={},n=this.querySelectorAll("template[data-astro-template]");for(let s of n){let i=s.closest(this.tagName);!i||!i.isSameNode(this)||(e[s.getAttribute("data-astro-template")||"default"]=s.innerHTML,s.remove())}for(let s of r){let i=s.closest(this.tagName);!i||!i.isSameNode(this)||(e[s.getAttribute("name")||"default"]=s.innerHTML)}let a=this.hasAttribute("props")?JSON.parse(this.getAttribute("props"),o):{};this.hydrator(this)(this.Component,a,e,{client:this.getAttribute("client")}),this.removeAttribute("ssr"),window.removeEventListener("astro:hydrate",this.hydrate),window.dispatchEvent(new CustomEvent("astro:hydrate"))}}connectedCallback(){!this.hasAttribute("await-children")||this.firstChild?this.childrenConnectedCallback():new MutationObserver((r,e)=>{e.disconnect(),setTimeout(()=>this.childrenConnectedCallback(),0)}).observe(this,{childList:!0})}async childrenConnectedCallback(){window.addEventListener("astro:hydrate",this.hydrate);let r=this.getAttribute("before-hydration-url");r&&await import(r),this.start()}start(){let r=JSON.parse(this.getAttribute("opts")),e=this.getAttribute("client");if(Astro[e]===void 0){window.addEventListener(`astro:${e}`,()=>this.start(),{once:!0});return}Astro[e](async()=>{let n=this.getAttribute("renderer-url"),[a,{default:l}]=await Promise.all([import(this.getAttribute("component-url")),n?import(n):()=>()=>{}]),s=this.getAttribute("component-export")||"default";if(!s.includes("."))this.Component=a[s];else{this.Component=a;for(let i of s.split("."))this.Component=this.Component[i]}return this.hydrator=l,this.hydrate},r,this)}attributeChangedCallback(){this.hydrate()}},c.observedAttributes=["props"],c))}})();</script><astro-island uid="aIFOr" component-url="/_astro/NavSidebar.63e48d0f.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{}" ssr="" client="only" opts="{&quot;name&quot;:&quot;NavSidebar&quot;,&quot;value&quot;:&quot;react&quot;}"></astro-island>
</div>
        <div class="flex-1">
            <main style="max-width: min(100ch, 100% - 4rem)" class="mx-auto">
                <h2 class="text-3xl font-bold pt-4">The Regulatory Genome, Gene Regulatory Networks and Transcriptomics</h2>
                <h3 class="text-2xl font-bold py-8">Research Summary</h3>
                <div class="grid gap-2">
        <script>(()=>{var e=async t=>{await(await t())()};(self.Astro||(self.Astro={})).load=e;window.dispatchEvent(new Event("astro:load"));})();</script><astro-island uid="Z20A1r5" prefix="r12" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;Eric Davidson&#39;s Regulatory Genome for Computer Science: Causality, Logic, and Proof Principles of the Genomic cis-Regulatory Code&quot;],&quot;blurb&quot;:[0,&quot;In this article, we discuss several computer science problems,\n                inspired by our 15-year-long collaboration with Prof. Eric Davidson,\n                focusing on computer science contributions to the study of the\n                regulatory genome. Our joint study was inspired by his lifetime\n                trailblazing research program rooted in causal gene regulatory\n                networks (GRNs), system completeness, genomic Boolean logic, and\n                genomically encoded regulatory information. We present first four\n                inspiring questions that Eric Davidson asked, and the follow-up,\n                namely, seven technical problems,&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">Eric Davidson&#x27;s Regulatory Genome for Computer Science: Causality, Logic, and Proof Principles of the Genomic cis-Regulatory Code</h3>In this article, we discuss several computer science problems,
                inspired by our 15-year-long collaboration with Prof. Eric Davidson,
                focusing on computer science contributions to the study of the
                regulatory genome. Our joint study was inspired by his lifetime
                trailblazing research program rooted in causal gene regulatory
                networks (GRNs), system completeness, genomic Boolean logic, and
                genomically encoded regulatory information. We present first four
                inspiring questions that Eric Davidson asked, and the follow-up,
                namely, seven technical problems,<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
            <p>
                In this article, we discuss several computer science problems,
                inspired by our 15-year-long collaboration with Prof. Eric
                Davidson, focusing on computer science contributions to the
                study of the regulatory genome. Our joint study was inspired by
                his lifetime trailblazing research program rooted in causal gene
                regulatory networks (GRNs), system completeness, genomic Boolean
                logic, and genomically encoded regulatory information. We
                present first four inspiring questions that Eric Davidson asked,
                and the follow-up, namely, seven technical problems, fully or
                partially resolved with the methods of computer science. At the
                center, and unifying the intellectual backbone of those
                technical challenges, stands “Causality.” Our collaboration
                produced the causality-inferred cisGRN-Lexicon database,
                containing the cis-regulatory architecture (CRA) of 600+
                transcription factor (TF)-encoding genes and other regulatory
                genes, in eight species: human, mouse, fruit fly, sea urchin,
                nematode, rat, chicken, and zebrafish. These CRAs are
                causality-inferred regulatory regions of genes, derived
                experimentally through the experimental method called
                “cis-regulatory analysis” (also known as the “Davidson
                criteria”). In this research program, causality challenges for
                computer science show up in two components: (1) how to define
                data structures that represent the causality-inferred, by the
                Davidson criteria, DNA structure data and to define a versatile
                software system to host them; and (2) how to identify by
                automated software for text analysis the experimental technical
                articles applying the Davidson criteria to the analysis to
                genes. We next present the cisGRN-Lexicon Meta-Analysis (Part
                I). We conclude the article with some reflections on
                epistemology and philosophy themes concerning the role of
                causality, logic, and proof in the emerging elegant mathematical
                theory and practice of the regulatory genome.
            </p>
            <p>
                It is challenging to explain what “explanation” is, and to
                understand what “understanding” is, when the technical task is
                to “prove” system-level causality completeness of a 50-gene
                causal GRN. Within the Peter-Davidson Boolean GRN model, the
                Peter-Davidson completeness “theorem” provides a seminal answer:
                Experimental causality system completeness = Computational exact
                prediction completeness.
            </p>
            <p>
                The article is organized as follows. Section 2 is dedicated to
                our Prof. Eric Davidson. Section 3 gives a brief introduction
                for computer scientists to the regulatory genome and its
                information processing operations in terms similar to the
                electronic computer. Section 4 proposes to honor Eric Davidson's
                life-long scientific work on the regulatory genome by naming a
                most fundamental time unit constant after him. Section 5
                presents four grand challenge questions that Eric Davidson
                asked, and seven follow-up problems inspired by the first two
                questions, which we fully or partially solved together. Central
                to the mentioned solutions is our construction of the
                cisGRN-Lexcion, the database of causally inferred CRA of 600+
                regulatory genes in eight species. Section 6 presents Part I of
                the cisGRN-Lexcion Meta-Analysis, coached as “rules” of the
                genomic cis-regulatory code. Section 7 is devoted to reflections
                on epistemological and philosophical themes: causality, logic,
                and proof in the elegant mathematical modeling of the regulatory
                genome. We present here the “Davidsonian Causal Systems Biology
                Axioms,” which guide us toward understanding of the meaning of
                “proving” causality completeness, for a complex experimental
                system, by exact computational predictions.
            </p>
        </template></astro-island>
        <astro-island uid="2t7gMl" prefix="r13" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;How Does the Regulatory Genome Work?&quot;],&quot;blurb&quot;:[0,&quot;The regulatory genome controls genome activity throughout the life\n            of an organism. This requires that complex information processing\n            functions are encoded in, and operated by, the regulatory genome.\n            Although much remains to be learned about how the regulatory genome\n            works, we here discuss two cases where regulatory functions have\n            been experimentally dissected in great detail and at the systems\n            level, and formalized by computational logic models.&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">How Does the Regulatory Genome Work?</h3>The regulatory genome controls genome activity throughout the life
            of an organism. This requires that complex information processing
            functions are encoded in, and operated by, the regulatory genome.
            Although much remains to be learned about how the regulatory genome
            works, we here discuss two cases where regulatory functions have
            been experimentally dissected in great detail and at the systems
            level, and formalized by computational logic models.<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
            <p>
                The regulatory genome controls genome activity throughout the
                life of an organism. This requires that complex information
                processing functions are encoded in, and operated by, the
                regulatory genome. Although much remains to be learned about how
                the regulatory genome works, we here discuss two cases where
                regulatory functions have been experimentally dissected in great
                detail and at the systems level, and formalized by computational
                logic models. Both examples derive from the sea urchin embryo,
                but assess two distinct organizational levels of genomic
                information processing. The first example shows how the
                regulatory system of a single gene, endo16, executes logic
                operations through individual transcription factor binding sites
                and cis-regulatory modules that control the expression of this
                gene. The second example shows information processing at the
                gene regulatory network (GRN) level. The GRN controlling
                development of the sea urchin endomesoderm has been
                experimentally explored at an almost complete level. A Boolean
                logic model of this GRN suggests that the modular logic
                functions encoded at the single-gene level show compositionality
                and suffice to account for integrated function at the network
                level. We discuss these examples both from a
                biological-experimental point of view and from a computer
                science-informational point of view, as both illuminate
                principles of how the regulatory genome works.
            </p>
        </template></astro-island>
        <astro-island uid="Z1rslGy" prefix="r14" component-url="/_astro/Abstract.2415f2d6.js" component-export="default" renderer-url="/_astro/client.4db56182.js" props="{&quot;title&quot;:[0,&quot;OCR-based image features for biomedical image and article classification: identifying documents relevant to cis-regulatory elements&quot;],&quot;blurb&quot;:[0,&quot;Images form a significant and useful source of information in\n            published biomedical articles, which is still under-utilized in\n            biomedical document classification and retrieval. Much current\n            work on biomedical image retrieval and classification employs\n            simple, standard image features such as gray scale histograms\n            and edge direction to represent and classify images. We have\n            used such features as well to classify images in our early work\n            [5], where we used image-class-tags to represent and classify\n            articles.&quot;]}" ssr="" client="load" opts="{&quot;name&quot;:&quot;Abstract&quot;,&quot;value&quot;:true}" await-children=""><div class="grid gap-2"><h3 class="text-lg font-bold" style="text-wrap:balance">OCR-based image features for biomedical image and article classification: identifying documents relevant to cis-regulatory elements</h3>Images form a significant and useful source of information in
            published biomedical articles, which is still under-utilized in
            biomedical document classification and retrieval. Much current
            work on biomedical image retrieval and classification employs
            simple, standard image features such as gray scale histograms
            and edge direction to represent and classify images. We have
            used such features as well to classify images in our early work
            [5], where we used image-class-tags to represent and classify
            articles.<button class="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium css-79xub" tabindex="0" type="button">See more</button></div><template data-astro-template>
            <p>
                Images form a significant and useful source of information in
                published biomedical articles, which is still under-utilized in
                biomedical document classification and retrieval. Much current
                work on biomedical image retrieval and classification employs
                simple, standard image features such as gray scale histograms
                and edge direction to represent and classify images. We have
                used such features as well to classify images in our early work
                [5], where we used image-class-tags to represent and classify
                articles.
            </p>
            <p>
                In the work presented here we focus on a different literature
                classification task, motivated by the need to identify articles
                discussing cis-regulatory elements and modules in the context of
                understanding complex gene-networks. The curators who try to
                identify such articles in the vast literature use as a major cue
                a certain type of image in which the conserved cis-regulatory
                region on the DNA is shown. Our experiments show that
                automatically identifying such images using common image
                features (like those mentioned above) can be highly error prone.
                However, using Optical Character Recognition (OCR) to extract
                alphabet characters from images, calculating character
                distribution and using the distribution parameters as image
                features, allows us to form a novel representation of images,
                and identify DNA-content in images with high precision and
                recall (over 0.9). Utilizing the occurrence of such DNA-rich
                images within articles, we train a classifier that identifies
                articles pertaining to cis-regulatory elements with a similarly
                high precision and recall. The use of OCR-based image features
                has much potential beyond the current task, to identify other
                types of biomedical sequence-based images showing DNA, RNA and
                proteins. Moreover, the ability to automatically identify such
                images has much potential to be widely applicable in other
                important biomedical document classification tasks.
            </p>
        </template></astro-island>
    </div>
                
                

                <h3 class="text-2xl font-bold py-8">Relevant Papers</h3>
                <ul class="grid gap-4">
        <li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Eric Davidson&#x27;s Regulatory Genome for Computer Science: Causality, Logic, and Proof Principles of the Genomic cis-Regulatory Code</h3><span class="text-md font-bold">2019</span></div><p class="py-2">Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Journal of Computational Biology</i></span>, vol. 26, no. 7</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="https://www.liebertpub.com/doi/epub/10.1089/cmb.2019.0144" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">How Does the Regulatory Genome Work?</h3><span class="text-md font-bold">2019</span></div><p class="py-2">Sorin Istrail, Isabelle S. Peter</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Journal of Computational Biology</i></span>, vol. 26, no. 7, pg. 685-695</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="https://www.liebertpub.com/doi/epub/10.1089/cmb.2019.0097" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">A quantitative reference transcriptome for Nematostella vectensis early embryonic development: a pipeline for de novo assembly in emerging model systems</h3><span class="text-md font-bold">2013</span></div><p class="py-2">Sarah Tulin, Derek Aguiar, Sorin Istrail, Joel Smith</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>EvoDevo</i></span>, vol. 4, no. 16</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.evodevojournal.com/content/4/1/16/abstract" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">OCR-based Image Features for Biomedical Image and Article Classification: Identifying Documents Relevant to Cis-Regulatory Elements</h3><span class="text-md font-bold">2012</span></div><p class="py-2">Hagit Shatkay, Ramya Narayanaswamy, Santosh Nagaral, Na Harrington, Dorothea Blostein, Ryan Tarpine, Kyle Schutter, Rohith Mv, Gowri Somanath, Sorin Istrail, Chandra Kambahmettu</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>ACM BCB</i></span></p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="https://dl.acm.org/doi/pdf/10.1145/2382936.2382949" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Functional cis-regulatory genomics for systems biology</h3><span class="text-md font-bold">2010</span></div><p class="py-2">Jongmin Nam, Ping Dong, Ryan Tarpine, Sorin Istrail, Eric H. Davidson</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Proceedings of the National Academy of Sciences</i></span>, vol. 107, no. 8, pg. 3930-3935</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/PNAS-2010-Nam-3930-5.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Practical Computational Methods for Regulatory Genomics: A cisGRN-Lexicon and cisGRN-Browser for Gene Regulatory Networks</h3><span class="text-md font-bold">2010</span></div><p class="py-2">Sorin Istrail, Ryan Tarpine, Kyle Schutter, Derek Aguiar</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>Humana Press</p><p><span>In <i>Computational Biology of Transcription Factor Binding</i></span>, vol. 674, pg. 369-399</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/praccompmethods.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The regulatory genome and the computer</h3><span class="text-md font-bold">2007</span></div><p class="py-2">Sorin Istrail, Smadar Ben-Tabou De-Leon, Eric H. Davidson</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Developmental Biology</i></span>, vol. 310, no. 2, pg. 187-195</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/1-s2.0-S0012160607012547-main.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The Transcriptome of the Sea Urchin Embryo</h3><span class="text-md font-bold">2006</span></div><p class="py-2">Manoj P. Samanta, Waraporn Tongprasit, Sorin Istrail, R. Andrew Cameron, Qiang Tu, Eric H. Davidson, Viktor Stolc</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Science</i></span>, vol. 314, no. 5801, pg. 960-962</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/transcSeaUrchin.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The Genome of the Sea Urchin Strongylocentrotus purpuratus</h3><span class="text-md font-bold">2006</span></div><p class="py-2">Sea Urchin Genome Sequencing Consortium, Erica Sodergren, George M. Weinstock, Eric H Davidson, R. Andrew Cameron, Richard A. Gibbs, Robert C. Angerer, Lynne M. Angerer, Maria Ina Arnone, David R. Burgess, Robert D. Burke, James A. Coffman, Michael Dean, Maurice R. Elphick, Charles A. Ettensohn, Kathy R. Foltz, Amro Hamdoun, Richard O. Hynes, William H. Klein, William Marzluff, David R. McClay, Robert L. Morris, Arcady Mushegian, Jonathan P. Rast, L. Courtney Smith, Michael C. Thorndyke, Victor D. Vacquier, Gary M. Wessel, Greg Wray, Lan Zhang, Christine G. Elsik, Olga Ermolaeva, Wratko Hlavina, Gretchen Hofmann, Paul Kitts, Melissa J. Landrum, Aaron J. Mackey, Donna Maglott, Georgia Panopoulou, Albert J. Poustka, Kim Pruitt, Victor Sapojnikov, Xingzhi Song, Alexandre Souvorov, Victor Solovyev, Zheng Wei, Charles A. Whittaker, Kim Worley, K. James Durbin, Yufeng Shen, Olivier Fedrigo, David Garfield, Ralph Haygood, Alexander Primus, Rahul Satija, Tonya Severson, Manuel L. Gonzalez-Garay, Andrew R. Jackson, Aleksandar Milosavljevic, Mark Tong, Christopher E. Killian, Brian T. Livingston, Fred H. Wilt, Nikki Adams, Robert Bell, Seth Carbonneau, Rocky Cheung, Patrick Cormier, Bertrand Cosson, Jenifer Croce, Antonio Fernandez-Guerra, Anne-Marie Genevire, Manisha Goel, Hemant Kelkar, Julia Morales, Odile Mulner-Lorillon, Anthony J. Robertson, Jared V. Goldstone, Bryan Cole, David Epel, Bert Gold, Mark E. Hahn, Meredith Howard-Ashby, Mark Scally, John J. Stegeman, Erin L. Allgood, Jonah Cool, Kyle M. Judkins, Shawn S. McCafferty, Ashlan M. Musante, Robert A. Obar, Amanda P. Rawson, Blair J. Rossetti, Ian R. Gibbons, Matthew P. Hoffman, Andrew Leone, Sorin Istrail, Stefan C. Materna, Manoj P. Samanta, Viktor et al. Stolc</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Science</i></span>, vol. 314, no. 5801, pg. 941-952</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/sci.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Logic functions of the genomic cis-regulatory code</h3><span class="text-md font-bold">2005</span></div><p class="py-2">Sorin Istrail, Eric Davidson</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Proceedings of the National Academy of Sciences</i></span>, vol. 102, no. 14, pg. 4954-4959</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/PNAS-2005-Istrail-4954-9.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">The Sequence of the Human Genome</h3><span class="text-md font-bold">2001</span></div><p class="py-2">J. Craig Venter, Mark D. Adams, Eugene W. Myers, Peter W. Li, Richard J. Mural, Granger G. Sutton, Hamilton O. Smith, Mark Yandell, Cheryl A. Evans, Robert A. Holt, Jeannine D. Gocayne, Peter Amanatides, Richard M. Ballew, Daniel H. Huson, Jennifer Russo Wortman, Qing Zhang, Chinnappa D. Kodira, Xiangqun H. Zheng, Lin Chen, Marian Skupski, Gangadharan Subramanian, Paul D. Thomas, Jinghui Zhang, George L. Gabor Miklos, Catherine Nelson, Samuel Broder, Andrew G. Clark, Joe Nadeau, Victor A. McKusick, Norton Zinder, Arnold J. Levine, Richard J. Roberts, Mel Simon, Carolyn Slayman, Michael Hunkapiller, Randall Bolanos, Arthur Delcher, Ian Dew, Daniel Fasulo, Michael Flanigan, Liliana Florea, Aaron Halpern, Sridhar Hannenhalli, Saul Kravitz, Samuel Levy, Clark Mobarry, Knut Reinert, Karin Remington, Jane Abu-Threideh, Ellen Beasley, Kendra Biddick, Vivien Bonazzi, Rhonda Brandon, Michele Cargill, Ishwar Chandramouliswaran, Rosane Charlab, Kabir Chaturvedi, Zuoming Deng, Valentina Di Francesco, Patrick Dunn, Karen Eilbeck, Carlos Evangelista, Andrei E. Gabrielian, Weiniu Gan, Wangmao Ge, Fangcheng Gong, Zhiping Gu, Ping Guan, Thomas J. Heiman, Maureen E. Higgins, Rui-Ru Ji, Zhaoxi Ke, Karen A. Ketchum, Zhongwu Lai, Yiding Lei, Zhenya Li, Jiayin Li, Yong Liang, Xiaoying Lin, Fu Lu, Gennady V. Merkulov, Natalia Milshina, Helen M. Moore, Ashwinikumar K Naik, Vaibhav A. Narayan, Beena Neelam, Deborah Nusskern, Douglas B. Rusch, Steven Salzberg, Wei Shao, Bixiong Shue, Jingtao Sun, Zhen Yuan Wang, Aihui Wang, Xin Wang, Jian Wang, Ming-Hui Wei, Ron Wides, Chunlin Xiao, Chunhua et al. Yan</p><div class="flex gap-6 justify-between"><div class="text-sm"><p><span>In <i>Science</i></span>, vol. 291, no. 5507, pg. 1304-1351</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/Science_2001_VenterThe_sequence_of_the_human.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li><li>
                    <div class="border-2 p-4 shadow-lg"><div class="flex gap-6 justify-between"><h3 class="text-xl" style="text-wrap:balance">Visualization challenges for a new cyberpharmaceutical computing paradigm</h3><span class="text-md font-bold">2001</span></div><p class="py-2">Russell J. Turner, Kabir Chaturvedi, Nathan J. Edwards, Daniel Fasulo, Aaron L. Halpern, Daniel H. Huson, Oliver Kohlbacher, Jason R. Miller, Knut Reinert, Karin A. Remington, Russell Schwartz, Brian Walenz, Shibu Yooseph, Sorin Istrail</p><div class="flex gap-6 justify-between"><div class="text-sm"><p>IEEE Press</p><p><span>In <i>IEEE 2001 symposium on parallel and large-data visualization and graphics</i></span>, pg. 7-18</p></div><div class="flex gap-4 items-center"><a class="text-2xl hover:fill-red-500" href="http://www.brown.edu/Research/Istrail_Lab/archive/papers/p7-turner.pdf" target="_blank" rel="noopener noreferrer"><svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 512 512"><path d="M64 464H96v48H64c-35.3 0-64-28.7-64-64V64C0 28.7 28.7 0 64 0H229.5c17 0 33.3 6.7 45.3 18.7l90.5 90.5c12 12 18.7 28.3 18.7 45.3V288H336V160H256c-17.7 0-32-14.3-32-32V48H64c-8.8 0-16 7.2-16 16V448c0 8.8 7.2 16 16 16zM176 352h32c30.9 0 56 25.1 56 56s-25.1 56-56 56H192v32c0 8.8-7.2 16-16 16s-16-7.2-16-16V448 368c0-8.8 7.2-16 16-16zm32 80c13.3 0 24-10.7 24-24s-10.7-24-24-24H192v48h16zm96-80h32c26.5 0 48 21.5 48 48v64c0 26.5-21.5 48-48 48H304c-8.8 0-16-7.2-16-16V368c0-8.8 7.2-16 16-16zm32 128c8.8 0 16-7.2 16-16V400c0-8.8-7.2-16-16-16H320v96h16zm80-112c0-8.8 7.2-16 16-16h48c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v32h32c8.8 0 16 7.2 16 16s-7.2 16-16 16H448v48c0 8.8-7.2 16-16 16s-16-7.2-16-16V432 368z"></path></svg></a></div></div></div>
                </li>
    </ul>
            </main>
        </div>
        <footer class="flex justify-center items-center p-4 bg-neutral text-neutral-content mt-20 text-sm">
    <div class="items-center grid-flow-col">
        <p>Copyright Istrail Lab © 2024 - All rights reserved</p>
    </div>
</footer>
    </body></html>